sys/Makefile.include

```diff
@@ -126,3 +126,7 @@ endif
 ifneq (,$(filter prng,$(USEMODULE)))
   include $(RIOTBASE)/sys/random/Makefile.include
 endif
+
+ifneq (,$(filter dbgpin,$(USEMODULE)))
+  INCLUDES += -include $(RIOTBASE)/sys/include/dbgpin.h
+endif
```

Comment:
For some reason this line causes issues with `hifive1b`, when removing and including the header manually it works well:

```
BOARD=hifive1b DBGPIN_PINS="GPIO_PIN(0,13),GPIO_PIN(0,12),GPIO_PIN(0,11)" make -C tests/dbgpin/ flash term -j3
```

![image](https://user-images.githubusercontent.com/23060007/94411766-3d892200-0179-11eb-99a6-6954e88fa4bb.png)

Otherwise I get multiple error messages in header files:

<details>

```
/home/francisco/opt/gnu-mcu-eclipse/riscv-none-gcc/8.2.0-2.2-20190521-0004/lib/gcc/riscv-none-embed/8.2.0/include/stddef.h: Assembler messages:
/home/francisco/opt/gnu-mcu-eclipse/riscv-none-gcc/8.2.0-2.2-20190521-0004/lib/gcc/riscv-none-embed/8.2.0/include/stddef.h:149: Error: unrecognized opcode `typedef int ptrdiff_t'
/home/francisco/opt/gnu-mcu-eclipse/riscv-none-gcc/8.2.0-2.2-20190521-0004/lib/gcc/riscv-none-embed/8.2.0/include/stddef.h:216: Error: unrecognized opcode `typedef unsigned int size_t'
/home/francisco/opt/gnu-mcu-eclipse/riscv-none-gcc/8.2.0-2.2-20190521-0004/lib/gcc/riscv-none-embed/8.2.0/include/stddef.h:328: Error: unrecognized opcode `typedef int wchar_t'
/home/francisco/opt/gnu-mcu-eclipse/riscv-none-gcc/8.2.0-2.2-20190521-0004/riscv-none-embed/include/machine/_default_types.h:41: Error: unrecognized opcode `typedef signed char __int8_t'
/home/francisco/opt/gnu-mcu-eclipse/riscv-none-gcc/8.2.0-2.2-20190521-0004/riscv-none-embed/include/machine/_default_types.h:43: Error: unrecognized opcode `typedef unsigned char __uint8_t'
/home/francisco/opt/gnu-mcu-eclipse/riscv-none-gcc/8.2.0-2.2-20190521-0004/riscv-none-embed/include/machine/_default_types.h:55: Error: unrecognized opcode `typedef short int __int16_t'
/home/francisco/opt/gnu-mcu-eclipse/riscv-none-gcc/8.2.0-2.2-20190521-0004/riscv-none-embed/include/machine/_default_types.h:57: Error: unrecognized opcode `typedef short unsigned int __uint16_t'
/home/francisco/opt/gnu-mcu-eclipse/riscv-none-gcc/8.2.0-2.2-20190521-0004/riscv-none-embed/include/machine/_default_types.h:77: Error: unrecognized opcode `typedef long int __int32_t'
/home/francisco/opt/gnu-mcu-eclipse/riscv-none-gcc/8.2.0-2.2-20190521-0004/riscv-none-embed/include/machine/_default_types.h:79: Error: unrecognized opcode `typedef long unsigned int __uint32_t'
/home/francisco/opt/gnu-mcu-eclipse/riscv-none-gcc/8.2.0-2.2-20190521-0004/riscv-none-embed/include/machine/_default_types.h:103: Error: unrecognized opcode `typedef long long int __int64_t'
/home/francisco/opt/gnu-mcu-eclipse/riscv-none-gcc/8.2.0-2.2-20190521-0004/riscv-none-embed/include/machine/_default_types.h:105: Error: unrecognized opcode `typedef long long unsigned int __uint64_t'
/home/francisco/opt/gnu-mcu-eclipse/riscv-none-gcc/8.2.0-2.2-20190521-0004/riscv-none-embed/include/machine/_default_types.h:134: Error: unrecognized opcode `typedef signed char __int_least8_t'
/home/francisco/opt/gnu-mcu-eclipse/riscv-none-gcc/8.2.0-2.2-20190521-0004/riscv-none-embed/include/machine/_default_types.h:136: Error: unrecognized opcode `typedef unsigned char __uint_least8_t'
/home/francisco/opt/gnu-mcu-eclipse/riscv-none-gcc/8.2.0-2.2-20190521-0004/riscv-none-embed/include/machine/_default_types.h:160: Error: unrecognized opcode `typedef short int __int_least16_t'
/home/francisco/opt/gnu-mcu-eclipse/riscv-none-gcc/8.2.0-2.2-20190521-0004/riscv-none-embed/include/machine/_default_types.h:162: Error: unrecognized opcode `typedef short unsigned int __uint_least16_t'
/home/francisco/opt/gnu-mcu-eclipse/riscv-none-gcc/8.2.0-2.2-20190521-0004/riscv-none-embed/include/machine/_default_types.h:182: Error: unrecognized opcode `typedef long int __int_least32_t'
/home/francisco/opt/gnu-mcu-eclipse/riscv-none-gcc/8.2.0-2.2-20190521-0004/riscv-none-embed/include/machine/_default_types.h:184: Error: unrecognized opcode `typedef long unsigned int __uint_least32_t'
/home/francisco/opt/gnu-mcu-eclipse/riscv-none-gcc/8.2.0-2.2-20190521-0004/riscv-none-embed/include/machine/_default_types.h:200: Error: unrecognized opcode `typedef long long int __int_least64_t'
/home/francisco/opt/gnu-mcu-eclipse/riscv-none-gcc/8.2.0-2.2-20190521-0004/riscv-none-embed/include/machine/_default_types.h:202: Error: unrecognized opcode `typedef long long unsigned int __uint_least64_t'
/home/francisco/opt/gnu-mcu-eclipse/riscv-none-gcc/8.2.0-2.2-20190521-0004/riscv-none-embed/include/machine/_default_types.h:214: Error: unrecognized opcode `typedef long long int __intmax_t'
/home/francisco/opt/gnu-mcu-eclipse/riscv-none-gcc/8.2.0-2.2-20190521-0004/riscv-none-embed/include/machine/_default_types.h:222: Error: unrecognized opcode `typedef long long unsigned int __uintmax_t'
/home/francisco/opt/gnu-mcu-eclipse/riscv-none-gcc/8.2.0-2.2-20190521-0004/riscv-none-embed/include/machine/_default_types.h:230: Error: unrecognized opcode `typedef int __intptr_t'
/home/francisco/opt/gnu-mcu-eclipse/riscv-none-gcc/8.2.0-2.2-20190521-0004/riscv-none-embed/include/machine/_default_types.h:232: Error: unrecognized opcode `typedef unsigned int __uintptr_t'
/home/francisco/opt/gnu-mcu-eclipse/riscv-none-gcc/8.2.0-2.2-20190521-0004/riscv-none-embed/include/sys/_stdint.h:20: Error: unrecognized opcode `typedef __int8_t int8_t'
/home/francisco/opt/gnu-mcu-eclipse/riscv-none-gcc/8.2.0-2.2-20190521-0004/riscv-none-embed/include/sys/_stdint.h:24: Error: unrecognized opcode `typedef __uint8_t uint8_t'
/home/francisco/opt/gnu-mcu-eclipse/riscv-none-gcc/8.2.0-2.2-20190521-0004/riscv-none-embed/include/sys/_stdint.h:32: Error: unrecognized opcode `typedef __int16_t int16_t'
/home/francisco/opt/gnu-mcu-eclipse/riscv-none-gcc/8.2.0-2.2-20190521-0004/riscv-none-embed/include/sys/_stdint.h:36: Error: unrecognized opcode `typedef __uint16_t uint16_t'
/home/francisco/opt/gnu-mcu-eclipse/riscv-none-gcc/8.2.0-2.2-20190521-0004/riscv-none-embed/include/sys/_stdint.h:44: Error: unrecognized opcode `typedef __int32_t int32_t'
/home/francisco/opt/gnu-mcu-eclipse/riscv-none-gcc/8.2.0-2.2-20190521-0004/riscv-none-embed/include/sys/_stdint.h:48: Error: unrecognized opcode `typedef __uint32_t uint32_t'
/home/francisco/opt/gnu-mcu-eclipse/riscv-none-gcc/8.2.0-2.2-20190521-0004/riscv-none-embed/include/sys/_stdint.h:56: Error: unrecognized opcode `typedef __int64_t int64_t'
/home/francisco/opt/gnu-mcu-eclipse/riscv-none-gcc/8.2.0-2.2-20190521-0004/riscv-none-embed/include/sys/_stdint.h:60: Error: unrecognized opcode `typedef __uint64_t uint64_t'
/home/francisco/opt/gnu-mcu-eclipse/riscv-none-gcc/8.2.0-2.2-20190521-0004/riscv-none-embed/include/sys/_stdint.h:67: Error: unrecognized opcode `typedef __intmax_t intmax_t'
/home/francisco/opt/gnu-mcu-eclipse/riscv-none-gcc/8.2.0-2.2-20190521-0004/riscv-none-embed/include/sys/_stdint.h:72: Error: unrecognized opcode `typedef __uintmax_t uintmax_t'
/home/francisco/opt/gnu-mcu-eclipse/riscv-none-gcc/8.2.0-2.2-20190521-0004/riscv-none-embed/include/sys/_stdint.h:77: Error: unrecognized opcode `typedef __intptr_t intptr_t'
/home/francisco/opt/gnu-mcu-eclipse/riscv-none-gcc/8.2.0-2.2-20190521-0004/riscv-none-embed/include/sys/_stdint.h:82: Error: unrecognized opcode `typedef __uintptr_t uintptr_t'
/home/francisco/opt/gnu-mcu-eclipse/riscv-none-gcc/8.2.0-2.2-20190521-0004/riscv-none-embed/include/stdint.h:21: Error: unrecognized opcode `typedef __int_least8_t int_least8_t'
/home/francisco/opt/gnu-mcu-eclipse/riscv-none-gcc/8.2.0-2.2-20190521-0004/riscv-none-embed/include/stdint.h:22: Error: unrecognized opcode `typedef __uint_least8_t uint_least8_t'
/home/francisco/opt/gnu-mcu-eclipse/riscv-none-gcc/8.2.0-2.2-20190521-0004/riscv-none-embed/include/stdint.h:27: Error: unrecognized opcode `typedef __int_least16_t int_least16_t'
/home/francisco/opt/gnu-mcu-eclipse/riscv-none-gcc/8.2.0-2.2-20190521-0004/riscv-none-embed/include/stdint.h:28: Error: unrecognized opcode `typedef __uint_least16_t uint_least16_t'
/home/francisco/opt/gnu-mcu-eclipse/riscv-none-gcc/8.2.0-2.2-20190521-0004/riscv-none-embed/include/stdint.h:33: Error: unrecognized opcode `typedef __int_least32_t int_least32_t'
/home/francisco/opt/gnu-mcu-eclipse/riscv-none-gcc/8.2.0-2.2-20190521-0004/riscv-none-embed/include/stdint.h:34: Error: unrecognized opcode `typedef __uint_least32_t uint_least32_t'
/home/francisco/opt/gnu-mcu-eclipse/riscv-none-gcc/8.2.0-2.2-20190521-0004/riscv-none-embed/include/stdint.h:39: Error: unrecognized opcode `typedef __int_least64_t int_least64_t'
/home/francisco/opt/gnu-mcu-eclipse/riscv-none-gcc/8.2.0-2.2-20190521-0004/riscv-none-embed/include/stdint.h:40: Error: unrecognized opcode `typedef __uint_least64_t uint_least64_t'
/home/francisco/opt/gnu-mcu-eclipse/riscv-none-gcc/8.2.0-2.2-20190521-0004/riscv-none-embed/include/stdint.h:51: Error: unrecognized opcode `typedef int int_fast8_t'
/home/francisco/opt/gnu-mcu-eclipse/riscv-none-gcc/8.2.0-2.2-20190521-0004/riscv-none-embed/include/stdint.h:52: Error: unrecognized opcode `typedef unsigned int uint_fast8_t'
/home/francisco/opt/gnu-mcu-eclipse/riscv-none-gcc/8.2.0-2.2-20190521-0004/riscv-none-embed/include/stdint.h:61: Error: unrecognized opcode `typedef int int_fast16_t'
/home/francisco/opt/gnu-mcu-eclipse/riscv-none-gcc/8.2.0-2.2-20190521-0004/riscv-none-embed/include/stdint.h:62: Error: unrecognized opcode `typedef unsigned int uint_fast16_t'
/home/francisco/opt/gnu-mcu-eclipse/riscv-none-gcc/8.2.0-2.2-20190521-0004/riscv-none-embed/include/stdint.h:71: Error: unrecognized opcode `typedef int int_fast32_t'
/home/francisco/opt/gnu-mcu-eclipse/riscv-none-gcc/8.2.0-2.2-20190521-0004/riscv-none-embed/include/stdint.h:72: Error: unrecognized opcode `typedef unsigned int uint_fast32_t'
/home/francisco/opt/gnu-mcu-eclipse/riscv-none-gcc/8.2.0-2.2-20190521-0004/riscv-none-embed/include/stdint.h:81: Error: unrecognized opcode `typedef long long int int_fast64_t'
/home/francisco/opt/gnu-mcu-eclipse/riscv-none-gcc/8.2.0-2.2-20190521-0004/riscv-none-embed/include/stdint.h:82: Error: unrecognized opcode `typedef long long unsigned int uint_fast64_t'
/home/francisco/opt/gnu-mcu-eclipse/riscv-none-gcc/8.2.0-2.2-20190521-0004/riscv-none-embed/include/xlocale.h:9: Error: unrecognized opcode `struct __locale_t'
/home/francisco/opt/gnu-mcu-eclipse/riscv-none-gcc/8.2.0-2.2-20190521-0004/riscv-none-embed/include/xlocale.h:10: Error: unrecognized opcode `typedef struct __locale_t*locale_t'
/home/francisco/opt/gnu-mcu-eclipse/riscv-none-gcc/8.2.0-2.2-20190521-0004/riscv-none-embed/include/inttypes.h:312: Error: unrecognized opcode `typedef struct{'
/home/francisco/opt/gnu-mcu-eclipse/riscv-none-gcc/8.2.0-2.2-20190521-0004/riscv-none-embed/include/inttypes.h:313: Error: unrecognized opcode `intmax_t quot'
/home/francisco/opt/gnu-mcu-eclipse/riscv-none-gcc/8.2.0-2.2-20190521-0004/riscv-none-embed/include/inttypes.h:314: Error: unrecognized opcode `intmax_t rem'
/home/francisco/opt/gnu-mcu-eclipse/riscv-none-gcc/8.2.0-2.2-20190521-0004/riscv-none-embed/include/inttypes.h:315: Error: junk at end of line, first unrecognized character is `}'
/home/francisco/opt/gnu-mcu-eclipse/riscv-none-gcc/8.2.0-2.2-20190521-0004/riscv-none-embed/include/inttypes.h:317: Error: unrecognized opcode `struct _reent'
/home/francisco/opt/gnu-mcu-eclipse/riscv-none-gcc/8.2.0-2.2-20190521-0004/riscv-none-embed/include/inttypes.h:323: Error: unrecognized opcode `extern intmax_t imaxabs(intmax_t j)'
/home/francisco/opt/gnu-mcu-eclipse/riscv-none-gcc/8.2.0-2.2-20190521-0004/riscv-none-embed/include/inttypes.h:324: Error: unrecognized opcode `extern imaxdiv_t imaxdiv(intmax_t numer,intmax_t denomer)'
/home/francisco/opt/gnu-mcu-eclipse/riscv-none-gcc/8.2.0-2.2-20190521-0004/riscv-none-embed/include/inttypes.h:325: Error: unrecognized opcode `extern intmax_t strtoimax(const char*__restrict,char**__restrict,int)'
/home/francisco/opt/gnu-mcu-eclipse/riscv-none-gcc/8.2.0-2.2-20190521-0004/riscv-none-embed/include/inttypes.h:326: Error: unrecognized opcode `extern intmax_t _strtoimax_r(struct _reent*,const char*__restrict,char**__restrict,int)'
/home/francisco/opt/gnu-mcu-eclipse/riscv-none-gcc/8.2.0-2.2-20190521-0004/riscv-none-embed/include/inttypes.h:327: Error: unrecognized opcode `extern uintmax_t strtoumax(const char*__restrict,char**__restrict,int)'
/home/francisco/opt/gnu-mcu-eclipse/riscv-none-gcc/8.2.0-2.2-20190521-0004/riscv-none-embed/include/inttypes.h:328: Error: unrecognized opcode `extern uintmax_t _strtoumax_r(struct _reent*,const char*__restrict,char**__restrict,int)'
/home/francisco/opt/gnu-mcu-eclipse/riscv-none-gcc/8.2.0-2.2-20190521-0004/riscv-none-embed/include/inttypes.h:329: Error: unrecognized opcode `extern intmax_t wcstoimax(const wchar_t*__restrict,wchar_t**__restrict,int)'
/home/francisco/opt/gnu-mcu-eclipse/riscv-none-gcc/8.2.0-2.2-20190521-0004/riscv-none-embed/include/inttypes.h:330: Error: unrecognized opcode `extern intmax_t _wcstoimax_r(struct _reent*,const wchar_t*__restrict,wchar_t**__restrict,int)'
/home/francisco/opt/gnu-mcu-eclipse/riscv-none-gcc/8.2.0-2.2-20190521-0004/riscv-none-embed/include/inttypes.h:331: Error: unrecognized opcode `extern uintmax_t wcstoumax(const wchar_t*__restrict,wchar_t**__restrict,int)'
/home/francisco/opt/gnu-mcu-eclipse/riscv-none-gcc/8.2.0-2.2-20190521-0004/riscv-none-embed/include/inttypes.h:332: Error: unrecognized opcode `extern uintmax_t _wcstoumax_r(struct _reent*,const wchar_t*__restrict,wchar_t**__restrict,int)'
/home/francisco/opt/gnu-mcu-eclipse/riscv-none-gcc/8.2.0-2.2-20190521-0004/riscv-none-embed/include/inttypes.h:335: Error: unrecognized opcode `extern intmax_t strtoimax_l(const char*__restrict,char**_restrict,int,locale_t)'
/home/francisco/opt/gnu-mcu-eclipse/riscv-none-gcc/8.2.0-2.2-20190521-0004/riscv-none-embed/include/inttypes.h:336: Error: unrecognized opcode `extern uintmax_t strtoumax_l(const char*__restrict,char**_restrict,int,locale_t)'
/home/francisco/opt/gnu-mcu-eclipse/riscv-none-gcc/8.2.0-2.2-20190521-0004/riscv-none-embed/include/inttypes.h:337: Error: unrecognized opcode `extern intmax_t wcstoimax_l(const wchar_t*__restrict,wchar_t**_restrict,int,locale_t)'
/home/francisco/opt/gnu-mcu-eclipse/riscv-none-gcc/8.2.0-2.2-20190521-0004/riscv-none-embed/include/inttypes.h:338: Error: unrecognized opcode `extern uintmax_t wcstoumax_l(const wchar_t*__restrict,wchar_t**_restrict,int,locale_t)'
/home/francisco/workspace/RIOT2/core/include/list.h:40: Error: unrecognized opcode `typedef struct list_node{'
/home/francisco/workspace/RIOT2/core/include/list.h:41: Error: unrecognized opcode `struct list_node*next'
/home/francisco/workspace/RIOT2/core/include/list.h:42: Error: junk at end of line, first unrecognized character is `}'
/home/francisco/workspace/RIOT2/core/include/list.h:53: Error: unrecognized opcode `static inline void list_add(list_node_t*node,list_node_t*new_node)'
/home/francisco/workspace/RIOT2/core/include/list.h:54: Error: junk at end of line, first unrecognized character is `{'
/home/francisco/workspace/RIOT2/core/include/list.h:55: Error: unrecognized opcode `new_node->next =node->next'
/home/francisco/workspace/RIOT2/core/include/list.h:56: Error: unrecognized opcode `node->next =new_node'
/home/francisco/workspace/RIOT2/core/include/list.h:57: Error: junk at end of line, first unrecognized character is `}'
/home/francisco/workspace/RIOT2/core/include/list.h:67: Error: unrecognized opcode `static inline list_node_t*list_remove_head(list_node_t*list)'
/home/francisco/workspace/RIOT2/core/include/list.h:68: Error: junk at end of line, first unrecognized character is `{'
/home/francisco/workspace/RIOT2/core/include/list.h:69: Error: unrecognized opcode `list_node_t *head=list->next'
/home/francisco/workspace/RIOT2/core/include/list.h:71: Error: unrecognized opcode `if (head){'
/home/francisco/workspace/RIOT2/core/include/list.h:72: Error: unrecognized opcode `list->next =head->next'
/home/francisco/workspace/RIOT2/core/include/list.h:73: Error: junk at end of line, first unrecognized character is `}'
/home/francisco/workspace/RIOT2/core/include/list.h:74: Error: unrecognized opcode `return head'
/home/francisco/workspace/RIOT2/core/include/list.h:75: Error: junk at end of line, first unrecognized character is `}'
/home/francisco/workspace/RIOT2/core/include/list.h:86: Error: unrecognized opcode `static inline list_node_t*list_remove(list_node_t*list,list_node_t*node)'
/home/francisco/workspace/RIOT2/core/include/list.h:87: Error: junk at end of line, first unrecognized character is `{'
/home/francisco/workspace/RIOT2/core/include/list.h:88: Error: unrecognized opcode `while (list->next){'
/home/francisco/workspace/RIOT2/core/include/list.h:89: Error: unrecognized opcode `if (list->next==node){'
/home/francisco/workspace/RIOT2/core/include/list.h:90: Error: unrecognized opcode `list->next =node->next'
/home/francisco/workspace/RIOT2/core/include/list.h:91: Error: unrecognized opcode `return node'
/home/francisco/workspace/RIOT2/core/include/list.h:92: Error: junk at end of line, first unrecognized character is `}'
/home/francisco/workspace/RIOT2/core/include/list.h:93: Error: bad expression
/home/francisco/workspace/RIOT2/core/include/list.h:93: Error: junk at end of line, first unrecognized character is `n'
/home/francisco/workspace/RIOT2/core/include/list.h:94: Error: junk at end of line, first unrecognized character is `}'
/home/francisco/workspace/RIOT2/core/include/list.h:95: Error: unrecognized opcode `return list->next'
/home/francisco/workspace/RIOT2/core/include/list.h:96: Error: junk at end of line, first unrecognized character is `}'
/home/francisco/workspace/RIOT2/core/include/clist.h:103: Error: unrecognized opcode `typedef list_node_t clist_node_t'
/home/francisco/workspace/RIOT2/core/include/clist.h:114: Error: unrecognized opcode `static inline void clist_rpush(clist_node_t*list,clist_node_t*new_node)'
/home/francisco/workspace/RIOT2/core/include/clist.h:115: Error: junk at end of line, first unrecognized character is `{'
/home/francisco/workspace/RIOT2/core/include/clist.h:116: Error: unrecognized opcode `if (list->next){'
/home/francisco/workspace/RIOT2/core/include/clist.h:117: Error: unrecognized opcode `new_node->next =list->next->next'
/home/francisco/workspace/RIOT2/core/include/clist.h:118: Error: unrecognized opcode `list->next->next =new_node'
/home/francisco/workspace/RIOT2/core/include/clist.h:119: Error: junk at end of line, first unrecognized character is `}'
/home/francisco/workspace/RIOT2/core/include/clist.h:120: Error: unrecognized opcode `else {'
/home/francisco/workspace/RIOT2/core/include/clist.h:121: Error: unrecognized opcode `new_node->next =new_node'
/home/francisco/workspace/RIOT2/core/include/clist.h:122: Error: junk at end of line, first unrecognized character is `}'
/home/francisco/workspace/RIOT2/core/include/clist.h:123: Error: unrecognized opcode `list->next =new_node'
/home/francisco/workspace/RIOT2/core/include/clist.h:124: Error: junk at end of line, first unrecognized character is `}'
/home/francisco/workspace/RIOT2/core/include/clist.h:135: Error: unrecognized opcode `static inline void clist_lpush(clist_node_t*list,clist_node_t*new_node)'
/home/francisco/workspace/RIOT2/core/include/clist.h:136: Error: junk at end of line, first unrecognized character is `{'
/home/francisco/workspace/RIOT2/core/include/clist.h:137: Error: unrecognized opcode `if (list->next){'
/home/francisco/workspace/RIOT2/core/include/clist.h:138: Error: unrecognized opcode `new_node->next =list->next->next'
/home/francisco/workspace/RIOT2/core/include/clist.h:139: Error: unrecognized opcode `list->next->next =new_node'
/home/francisco/workspace/RIOT2/core/include/clist.h:140: Error: junk at end of line, first unrecognized character is `}'
/home/francisco/workspace/RIOT2/core/include/clist.h:141: Error: unrecognized opcode `else {'
/home/francisco/workspace/RIOT2/core/include/clist.h:142: Error: unrecognized opcode `new_node->next =new_node'
/home/francisco/workspace/RIOT2/core/include/clist.h:143: Error: unrecognized opcode `list->next =new_node'
/home/francisco/workspace/RIOT2/core/include/clist.h:144: Error: junk at end of line, first unrecognized character is `}'
/home/francisco/workspace/RIOT2/core/include/clist.h:145: Error: junk at end of line, first unrecognized character is `}'
/home/francisco/workspace/RIOT2/core/include/clist.h:155: Error: unrecognized opcode `static inline clist_node_t*clist_lpop(clist_node_t*list)'
/home/francisco/workspace/RIOT2/core/include/clist.h:156: Error: junk at end of line, first unrecognized character is `{'
/home/francisco/workspace/RIOT2/core/include/clist.h:157: Error: unrecognized opcode `if (list->next){'
/home/francisco/workspace/RIOT2/core/include/clist.h:158: Error: unrecognized opcode `clist_node_t *first=list->next->next'
/home/francisco/workspace/RIOT2/core/include/clist.h:159: Error: unrecognized opcode `if (list->next==first){'
/home/francisco/workspace/RIOT2/core/include/clist.h:160: Error: unrecognized opcode `list->next =((void*)0)'
/home/francisco/workspace/RIOT2/core/include/clist.h:161: Error: junk at end of line, first unrecognized character is `}'
/home/francisco/workspace/RIOT2/core/include/clist.h:162: Error: unrecognized opcode `else {'
/home/francisco/workspace/RIOT2/core/include/clist.h:163: Error: unrecognized opcode `list->next->next =first->next'
/home/francisco/workspace/RIOT2/core/include/clist.h:164: Error: junk at end of line, first unrecognized character is `}'
/home/francisco/workspace/RIOT2/core/include/clist.h:165: Error: unrecognized opcode `return first'
/home/francisco/workspace/RIOT2/core/include/clist.h:166: Error: junk at end of line, first unrecognized character is `}'
/home/francisco/workspace/RIOT2/core/include/clist.h:167: Error: unrecognized opcode `else {'
/home/francisco/workspace/RIOT2/core/include/clist.h:168: Error: unrecognized opcode `return ((void*)0)'
/home/francisco/workspace/RIOT2/core/include/clist.h:169: Error: junk at end of line, first unrecognized character is `}'
/home/francisco/workspace/RIOT2/core/include/clist.h:170: Error: junk at end of line, first unrecognized character is `}'
/home/francisco/workspace/RIOT2/core/include/clist.h:185: Error: unrecognized opcode `static inline void clist_lpoprpush(clist_node_t*list)'
/home/francisco/workspace/RIOT2/core/include/clist.h:186: Error: junk at end of line, first unrecognized character is `{'
/home/francisco/workspace/RIOT2/core/include/clist.h:187: Error: unrecognized opcode `if (list->next){'
/home/francisco/workspace/RIOT2/core/include/clist.h:188: Error: unrecognized opcode `list->next =list->next->next'
/home/francisco/workspace/RIOT2/core/include/clist.h:189: Error: junk at end of line, first unrecognized character is `}'
/home/francisco/workspace/RIOT2/core/include/clist.h:190: Error: junk at end of line, first unrecognized character is `}'
/home/francisco/workspace/RIOT2/core/include/clist.h:200: Error: unrecognized opcode `static inline clist_node_t*clist_lpeek(const clist_node_t*list)'
/home/francisco/workspace/RIOT2/core/include/clist.h:201: Error: junk at end of line, first unrecognized character is `{'
/home/francisco/workspace/RIOT2/core/include/clist.h:202: Error: unrecognized opcode `if (list->next){'
/home/francisco/workspace/RIOT2/core/include/clist.h:203: Error: unrecognized opcode `return list->next->next'
/home/francisco/workspace/RIOT2/core/include/clist.h:204: Error: junk at end of line, first unrecognized character is `}'
/home/francisco/workspace/RIOT2/core/include/clist.h:205: Error: unrecognized opcode `return ((void*)0)'
/home/francisco/workspace/RIOT2/core/include/clist.h:206: Error: junk at end of line, first unrecognized character is `}'
/home/francisco/workspace/RIOT2/core/include/clist.h:216: Error: unrecognized opcode `static inline clist_node_t*clist_rpeek(const clist_node_t*list)'
/home/francisco/workspace/RIOT2/core/include/clist.h:217: Error: junk at end of line, first unrecognized character is `{'
/home/francisco/workspace/RIOT2/core/include/clist.h:218: Error: unrecognized opcode `return list->next'
/home/francisco/workspace/RIOT2/core/include/clist.h:219: Error: junk at end of line, first unrecognized character is `}'
/home/francisco/workspace/RIOT2/core/include/clist.h:229: Error: unrecognized opcode `static inline clist_node_t*clist_rpop(clist_node_t*list)'
/home/francisco/workspace/RIOT2/core/include/clist.h:230: Error: junk at end of line, first unrecognized character is `{'
/home/francisco/workspace/RIOT2/core/include/clist.h:231: Error: unrecognized opcode `if (list->next){'
/home/francisco/workspace/RIOT2/core/include/clist.h:232: Error: unrecognized opcode `list_node_t *last=list->next'
/home/francisco/workspace/RIOT2/core/include/clist.h:233: Error: unrecognized opcode `while (list->next->next!=last){'
/home/francisco/workspace/RIOT2/core/include/clist.h:234: Error: unrecognized opcode `clist_lpoprpush(list)'
/home/francisco/workspace/RIOT2/core/include/clist.h:235: Error: junk at end of line, first unrecognized character is `}'
/home/francisco/workspace/RIOT2/core/include/clist.h:236: Error: unrecognized opcode `return clist_lpop(list)'
/home/francisco/workspace/RIOT2/core/include/clist.h:237: Error: junk at end of line, first unrecognized character is `}'
/home/francisco/workspace/RIOT2/core/include/clist.h:238: Error: unrecognized opcode `else {'
/home/francisco/workspace/RIOT2/core/include/clist.h:239: Error: unrecognized opcode `return ((void*)0)'
/home/francisco/workspace/RIOT2/core/include/clist.h:240: Error: junk at end of line, first unrecognized character is `}'
/home/francisco/workspace/RIOT2/core/include/clist.h:241: Error: junk at end of line, first unrecognized character is `}'
/home/francisco/workspace/RIOT2/core/include/clist.h:255: Error: unrecognized opcode `static inline clist_node_t*clist_find_before(const clist_node_t*list,'
/home/francisco/workspace/RIOT2/core/include/clist.h:256: Error: unrecognized opcode `const clist_node_t*node)'
/home/francisco/workspace/RIOT2/core/include/clist.h:257: Error: junk at end of line, first unrecognized character is `{'
/home/francisco/workspace/RIOT2/core/include/clist.h:258: Error: unrecognized opcode `clist_node_t *pos=list->next'
/home/francisco/workspace/RIOT2/core/include/clist.h:260: Error: unrecognized opcode `if (!pos){'
/home/francisco/workspace/RIOT2/core/include/clist.h:261: Error: unrecognized opcode `return ((void*)0)'
/home/francisco/workspace/RIOT2/core/include/clist.h:262: Error: junk at end of line, first unrecognized character is `}'
/home/francisco/workspace/RIOT2/core/include/clist.h:263: Error: unrecognized opcode `do {'
/home/francisco/workspace/RIOT2/core/include/clist.h:264: Error: bad expression
/home/francisco/workspace/RIOT2/core/include/clist.h:264: Error: junk at end of line, first unrecognized character is `n'
/home/francisco/workspace/RIOT2/core/include/clist.h:265: Error: unrecognized opcode `if (pos->next==node){'
/home/francisco/workspace/RIOT2/core/include/clist.h:266: Error: unrecognized opcode `return pos'
/home/francisco/workspace/RIOT2/core/include/clist.h:267: Error: junk at end of line, first unrecognized character is `}'
/home/francisco/workspace/RIOT2/core/include/clist.h:268: Error: junk at end of line, first unrecognized character is `}'
/home/francisco/workspace/RIOT2/core/include/clist.h:270: Error: unrecognized opcode `return ((void*)0)'
/home/francisco/workspace/RIOT2/core/include/clist.h:271: Error: junk at end of line, first unrecognized character is `}'
/home/francisco/workspace/RIOT2/core/include/clist.h:285: Error: unrecognized opcode `static inline clist_node_t*clist_find(const clist_node_t*list,'
/home/francisco/workspace/RIOT2/core/include/clist.h:286: Error: unrecognized opcode `const clist_node_t*node)'
/home/francisco/workspace/RIOT2/core/include/clist.h:287: Error: junk at end of line, first unrecognized character is `{'
/home/francisco/workspace/RIOT2/core/include/clist.h:288: Error: unrecognized opcode `clist_node_t *tmp=clist_find_before(list,node)'
/home/francisco/workspace/RIOT2/core/include/clist.h:290: Error: unrecognized opcode `if (tmp){'
/home/francisco/workspace/RIOT2/core/include/clist.h:291: Error: unrecognized opcode `return tmp->next'
/home/francisco/workspace/RIOT2/core/include/clist.h:292: Error: junk at end of line, first unrecognized character is `}'
/home/francisco/workspace/RIOT2/core/include/clist.h:293: Error: unrecognized opcode `else {'
/home/francisco/workspace/RIOT2/core/include/clist.h:294: Error: unrecognized opcode `return ((void*)0)'
/home/francisco/workspace/RIOT2/core/include/clist.h:295: Error: junk at end of line, first unrecognized character is `}'
/home/francisco/workspace/RIOT2/core/include/clist.h:296: Error: junk at end of line, first unrecognized character is `}'
/home/francisco/workspace/RIOT2/core/include/clist.h:310: Error: unrecognized opcode `static inline clist_node_t*clist_remove(clist_node_t*list,clist_node_t*node)'
/home/francisco/workspace/RIOT2/core/include/clist.h:311: Error: junk at end of line, first unrecognized character is `{'
/home/francisco/workspace/RIOT2/core/include/clist.h:312: Error: unrecognized opcode `if (list->next){'
/home/francisco/workspace/RIOT2/core/include/clist.h:313: Error: unrecognized opcode `if (list->next->next==node){'
/home/francisco/workspace/RIOT2/core/include/clist.h:314: Error: unrecognized opcode `return clist_lpop(list)'
/home/francisco/workspace/RIOT2/core/include/clist.h:315: Error: junk at end of line, first unrecognized character is `}'
/home/francisco/workspace/RIOT2/core/include/clist.h:316: Error: unrecognized opcode `else {'
/home/francisco/workspace/RIOT2/core/include/clist.h:317: Error: unrecognized opcode `clist_node_t *tmp=clist_find_before(list,node)'
/home/francisco/workspace/RIOT2/core/include/clist.h:318: Error: unrecognized opcode `if (tmp){'
/home/francisco/workspace/RIOT2/core/include/clist.h:319: Error: unrecognized opcode `tmp->next =tmp->next->next'
/home/francisco/workspace/RIOT2/core/include/clist.h:320: Error: unrecognized opcode `if (node==list->next){'
/home/francisco/workspace/RIOT2/core/include/clist.h:321: Error: unrecognized opcode `list->next =tmp'
/home/francisco/workspace/RIOT2/core/include/clist.h:322: Error: junk at end of line, first unrecognized character is `}'
/home/francisco/workspace/RIOT2/core/include/clist.h:323: Error: unrecognized opcode `return node'
/home/francisco/workspace/RIOT2/core/include/clist.h:324: Error: junk at end of line, first unrecognized character is `}'
/home/francisco/workspace/RIOT2/core/include/clist.h:325: Error: junk at end of line, first unrecognized character is `}'
/home/francisco/workspace/RIOT2/core/include/clist.h:326: Error: junk at end of line, first unrecognized character is `}'
/home/francisco/workspace/RIOT2/core/include/clist.h:328: Error: unrecognized opcode `return ((void*)0)'
/home/francisco/workspace/RIOT2/core/include/clist.h:329: Error: junk at end of line, first unrecognized character is `}'
/home/francisco/workspace/RIOT2/core/include/clist.h:346: Error: unrecognized opcode `static inline clist_node_t*clist_foreach(clist_node_t*list,int(*func)('
/home/francisco/workspace/RIOT2/core/include/clist.h:347: Error: unrecognized opcode `clist_node_t *,'
/home/francisco/workspace/RIOT2/core/include/clist.h:348: Error: unrecognized opcode `void *),void*arg)'
/home/francisco/workspace/RIOT2/core/include/clist.h:349: Error: junk at end of line, first unrecognized character is `{'
/home/francisco/workspace/RIOT2/core/include/clist.h:350: Error: unrecognized opcode `clist_node_t *node=list->next'
/home/francisco/workspace/RIOT2/core/include/clist.h:352: Error: unrecognized opcode `if (node){'
/home/francisco/workspace/RIOT2/core/include/clist.h:353: Error: unrecognized opcode `do {'
/home/francisco/workspace/RIOT2/core/include/clist.h:354: Error: bad expression
/home/francisco/workspace/RIOT2/core/include/clist.h:354: Error: junk at end of line, first unrecognized character is `n'
/home/francisco/workspace/RIOT2/core/include/clist.h:355: Error: unrecognized opcode `if (func(node,arg)){'
/home/francisco/workspace/RIOT2/core/include/clist.h:356: Error: unrecognized opcode `return node'
/home/francisco/workspace/RIOT2/core/include/clist.h:357: Error: junk at end of line, first unrecognized character is `}'
/home/francisco/workspace/RIOT2/core/include/clist.h:358: Error: junk at end of line, first unrecognized character is `}'
/home/francisco/workspace/RIOT2/core/include/clist.h:359: Error: junk at end of line, first unrecognized character is `}'
/home/francisco/workspace/RIOT2/core/include/clist.h:361: Error: unrecognized opcode `return ((void*)0)'
/home/francisco/workspace/RIOT2/core/include/clist.h:362: Error: junk at end of line, first unrecognized character is `}'
/home/francisco/workspace/RIOT2/core/include/clist.h:368: Error: unrecognized opcode `typedef int(*clist_cmp_func_t)(clist_node_t*a,clist_node_t*b)'
/home/francisco/workspace/RIOT2/core/include/clist.h:380: Error: unrecognized opcode `clist_node_t *_clist_sort(clist_node_t*list_head,clist_cmp_func_t cmp)'
/home/francisco/workspace/RIOT2/core/include/clist.h:424: Error: unrecognized opcode `static inline void clist_sort(clist_node_t*list,clist_cmp_func_t cmp)'
/home/francisco/workspace/RIOT2/core/include/clist.h:425: Error: junk at end of line, first unrecognized character is `{'
/home/francisco/workspace/RIOT2/core/include/clist.h:426: Error: unrecognized opcode `if (list->next){'
/home/francisco/workspace/RIOT2/core/include/clist.h:427: Error: unrecognized opcode `list->next =_clist_sort(list->next->next,cmp)'
/home/francisco/workspace/RIOT2/core/include/clist.h:428: Error: junk at end of line, first unrecognized character is `}'
/home/francisco/workspace/RIOT2/core/include/clist.h:429: Error: junk at end of line, first unrecognized character is `}'
/home/francisco/workspace/RIOT2/core/include/clist.h:438: Error: unrecognized opcode `static inline size_t clist_count(clist_node_t*list)'
/home/francisco/workspace/RIOT2/core/include/clist.h:439: Error: junk at end of line, first unrecognized character is `{'
/home/francisco/workspace/RIOT2/core/include/clist.h:440: Error: unrecognized opcode `clist_node_t *node=list->next'
/home/francisco/workspace/RIOT2/core/include/clist.h:441: Error: unrecognized opcode `size_t cnt=0'
/home/francisco/workspace/RIOT2/core/include/clist.h:443: Error: unrecognized opcode `if (node){'
/home/francisco/workspace/RIOT2/core/include/clist.h:444: Error: unrecognized opcode `do {'
/home/francisco/workspace/RIOT2/core/include/clist.h:445: Error: bad expression
/home/francisco/workspace/RIOT2/core/include/clist.h:445: Error: junk at end of line, first unrecognized character is `n'
/home/francisco/workspace/RIOT2/core/include/clist.h:446: Error: junk at end of line, first unrecognized character is `+'
/home/francisco/workspace/RIOT2/core/include/clist.h:447: Error: junk at end of line, first unrecognized character is `}'
/home/francisco/workspace/RIOT2/core/include/clist.h:448: Error: junk at end of line, first unrecognized character is `}'
/home/francisco/workspace/RIOT2/core/include/clist.h:450: Error: unrecognized opcode `return cnt'
/home/francisco/workspace/RIOT2/core/include/clist.h:451: Error: junk at end of line, first unrecognized character is `}'
/home/francisco/workspace/RIOT2/core/include/panic.h:34: Error: unrecognized opcode `typedef enum{'
/home/francisco/workspace/RIOT2/core/include/panic.h:35: Error: unrecognized opcode `panic_general_error,'
/home/francisco/workspace/RIOT2/core/include/panic.h:36: Error: unrecognized opcode `panic_soft_reboot,'
/home/francisco/workspace/RIOT2/core/include/panic.h:37: Error: unrecognized opcode `panic_hard_reboot,'
/home/francisco/workspace/RIOT2/core/include/panic.h:38: Error: unrecognized opcode `panic_assert_fail,'
/home/francisco/workspace/RIOT2/core/include/panic.h:39: Error: unrecognized opcode `panic_expect_fail,'
/home/francisco/workspace/RIOT2/core/include/panic.h:52: Error: unrecognized opcode `panic_ssp,'
/home/francisco/workspace/RIOT2/core/include/panic.h:53: Error: unrecognized opcode `panic_undefined'
/home/francisco/workspace/RIOT2/core/include/panic.h:54: Error: junk at end of line, first unrecognized character is `}'
/home/francisco/workspace/RIOT2/core/include/panic.h:76: Error: unrecognized opcode `__attribute__((noreturn)) void core_panic(core_panic_t crash_code,const char*message)'
/home/francisco/workspace/RIOT2/core/include/panic.h:84: Error: unrecognized opcode `void panic_arch(void)'
/home/francisco/workspace/RIOT2/core/include/assert.h:56: Error: unrecognized opcode `extern const char assert_crash_message[]'
/home/francisco/workspace/RIOT2/core/include/cib.h:34: Error: unrecognized opcode `typedef struct{'
/home/francisco/workspace/RIOT2/core/include/cib.h:35: Error: unrecognized opcode `unsigned int read_count'
/home/francisco/workspace/RIOT2/core/include/cib.h:36: Error: unrecognized opcode `unsigned int write_count'
/home/francisco/workspace/RIOT2/core/include/cib.h:37: Error: unrecognized opcode `unsigned int mask'
/home/francisco/workspace/RIOT2/core/include/cib.h:38: Error: junk at end of line, first unrecognized character is `}'
/home/francisco/workspace/RIOT2/core/include/cib.h:58: Error: unrecognized opcode `static inline void cib_init(cib_t*__restrict cib,unsigned int size)'
/home/francisco/workspace/RIOT2/core/include/cib.h:59: Error: junk at end of line, first unrecognized character is `{'
/home/francisco/workspace/RIOT2/core/include/cib.h:61: Error: junk at end of line, first unrecognized character is `('
/home/francisco/workspace/RIOT2/core/include/cib.h:63: Error: unrecognized opcode `cib_t c={0,0,(size)-1}'
/home/francisco/workspace/RIOT2/core/include/cib.h:64: Error: junk at end of line, first unrecognized character is `*'
/home/francisco/workspace/RIOT2/core/include/cib.h:65: Error: junk at end of line, first unrecognized character is `}'
/home/francisco/workspace/RIOT2/core/include/cib.h:74: Error: unrecognized opcode `static inline unsigned int cib_avail(const cib_t*cib)'
/home/francisco/workspace/RIOT2/core/include/cib.h:75: Error: junk at end of line, first unrecognized character is `{'
/home/francisco/workspace/RIOT2/core/include/cib.h:76: Error: unrecognized opcode `return cib->write_count-cib->read_count'
/home/francisco/workspace/RIOT2/core/include/cib.h:77: Error: junk at end of line, first unrecognized character is `}'
/home/francisco/workspace/RIOT2/core/include/cib.h:86: Error: unrecognized opcode `static inline unsigned int cib_full(const cib_t*cib)'
/home/francisco/workspace/RIOT2/core/include/cib.h:87: Error: junk at end of line, first unrecognized character is `{'
/home/francisco/workspace/RIOT2/core/include/cib.h:88: Error: unrecognized opcode `return ((int)cib_avail(cib))>((int)cib->mask)'
/home/francisco/workspace/RIOT2/core/include/cib.h:89: Error: junk at end of line, first unrecognized character is `}'
/home/francisco/workspace/RIOT2/core/include/cib.h:98: Error: unrecognized opcode `static inline int cib_get(cib_t*__restrict cib)'
/home/francisco/workspace/RIOT2/core/include/cib.h:99: Error: junk at end of line, first unrecognized character is `{'
/home/francisco/workspace/RIOT2/core/include/cib.h:100: Error: unrecognized opcode `if (cib_avail(cib)){'
/home/francisco/workspace/RIOT2/core/include/cib.h:101: Error: unrecognized opcode `return (int)(cib->read_count++&cib->mask)'
/home/francisco/workspace/RIOT2/core/include/cib.h:102: Error: junk at end of line, first unrecognized character is `}'
/home/francisco/workspace/RIOT2/core/include/cib.h:104: Error: unrecognized opcode `return -1'
/home/francisco/workspace/RIOT2/core/include/cib.h:105: Error: junk at end of line, first unrecognized character is `}'
/home/francisco/workspace/RIOT2/core/include/cib.h:114: Error: unrecognized opcode `static inline int cib_peek(cib_t*__restrict cib)'
/home/francisco/workspace/RIOT2/core/include/cib.h:115: Error: junk at end of line, first unrecognized character is `{'
/home/francisco/workspace/RIOT2/core/include/cib.h:116: Error: unrecognized opcode `if (cib_avail(cib)){'
/home/francisco/workspace/RIOT2/core/include/cib.h:117: Error: unrecognized opcode `return (int)(cib->read_count&cib->mask)'
/home/francisco/workspace/RIOT2/core/include/cib.h:118: Error: junk at end of line, first unrecognized character is `}'
/home/francisco/workspace/RIOT2/core/include/cib.h:120: Error: unrecognized opcode `return -1'
/home/francisco/workspace/RIOT2/core/include/cib.h:121: Error: junk at end of line, first unrecognized character is `}'
/home/francisco/workspace/RIOT2/core/include/cib.h:132: Error: unrecognized opcode `static inline int cib_get_unsafe(cib_t*cib)'
/home/francisco/workspace/RIOT2/core/include/cib.h:133: Error: junk at end of line, first unrecognized character is `{'
/home/francisco/workspace/RIOT2/core/include/cib.h:134: Error: unrecognized opcode `return (int)(cib->read_count++&cib->mask)'
/home/francisco/workspace/RIOT2/core/include/cib.h:135: Error: junk at end of line, first unrecognized character is `}'
/home/francisco/workspace/RIOT2/core/include/cib.h:144: Error: unrecognized opcode `static inline int cib_put(cib_t*__restrict cib)'
/home/francisco/workspace/RIOT2/core/include/cib.h:145: Error: junk at end of line, first unrecognized character is `{'
/home/francisco/workspace/RIOT2/core/include/cib.h:146: Error: unrecognized opcode `unsigned int avail=cib_avail(cib)'
/home/francisco/workspace/RIOT2/core/include/cib.h:149: Error: unrecognized opcode `if ((int)avail<=(int)cib->mask){'
/home/francisco/workspace/RIOT2/core/include/cib.h:150: Error: unrecognized opcode `return (int)(cib->write_count++&cib->mask)'
/home/francisco/workspace/RIOT2/core/include/cib.h:151: Error: junk at end of line, first unrecognized character is `}'
/home/francisco/workspace/RIOT2/core/include/cib.h:153: Error: unrecognized opcode `return -1'
/home/francisco/workspace/RIOT2/core/include/cib.h:154: Error: junk at end of line, first unrecognized character is `}'
/home/francisco/workspace/RIOT2/core/include/cib.h:165: Error: unrecognized opcode `static inline int cib_put_unsafe(cib_t*cib)'
/home/francisco/workspace/RIOT2/core/include/cib.h:166: Error: junk at end of line, first unrecognized character is `{'
/home/francisco/workspace/RIOT2/core/include/cib.h:167: Error: unrecognized opcode `return (int)(cib->write_count++&cib->mask)'
/home/francisco/workspace/RIOT2/core/include/cib.h:168: Error: junk at end of line, first unrecognized character is `}'
/home/francisco/opt/gnu-mcu-eclipse/riscv-none-gcc/8.2.0-2.2-20190521-0004/riscv-none-embed/include/sys/types.h:28: Error: unrecognized opcode `typedef __uint8_t u_int8_t'
/home/francisco/opt/gnu-mcu-eclipse/riscv-none-gcc/8.2.0-2.2-20190521-0004/riscv-none-embed/include/sys/types.h:31: Error: unrecognized opcode `typedef __uint16_t u_int16_t'
/home/francisco/opt/gnu-mcu-eclipse/riscv-none-gcc/8.2.0-2.2-20190521-0004/riscv-none-embed/include/sys/types.h:34: Error: unrecognized opcode `typedef __uint32_t u_int32_t'
/home/francisco/opt/gnu-mcu-eclipse/riscv-none-gcc/8.2.0-2.2-20190521-0004/riscv-none-embed/include/sys/types.h:37: Error: unrecognized opcode `typedef __uint64_t u_int64_t'
/home/francisco/opt/gnu-mcu-eclipse/riscv-none-gcc/8.2.0-2.2-20190521-0004/riscv-none-embed/include/sys/types.h:39: Error: unrecognized opcode `typedef int register_t'
/home/francisco/opt/gnu-mcu-eclipse/riscv-none-gcc/8.2.0-2.2-20190521-0004/riscv-none-embed/include/sys/lock.h:11: Error: unrecognized opcode `typedef int _LOCK_T'
/home/francisco/opt/gnu-mcu-eclipse/riscv-none-gcc/8.2.0-2.2-20190521-0004/riscv-none-embed/include/sys/lock.h:12: Error: unrecognized opcode `typedef int _LOCK_RECURSIVE_T'
/home/francisco/opt/gnu-mcu-eclipse/riscv-none-gcc/8.2.0-2.2-20190521-0004/riscv-none-embed/include/sys/_types.h:28: Error: unrecognized opcode `typedef long __blkcnt_t'
/home/francisco/opt/gnu-mcu-eclipse/riscv-none-gcc/8.2.0-2.2-20190521-0004/riscv-none-embed/include/sys/_types.h:32: Error: unrecognized opcode `typedef long __blksize_t'
/home/francisco/opt/gnu-mcu-eclipse/riscv-none-gcc/8.2.0-2.2-20190521-0004/riscv-none-embed/include/sys/_types.h:36: Error: unrecognized opcode `typedef __uint64_t __fsblkcnt_t'
/home/francisco/opt/gnu-mcu-eclipse/riscv-none-gcc/8.2.0-2.2-20190521-0004/riscv-none-embed/include/sys/_types.h:40: Error: unrecognized opcode `typedef __uint32_t __fsfilcnt_t'
/home/francisco/opt/gnu-mcu-eclipse/riscv-none-gcc/8.2.0-2.2-20190521-0004/riscv-none-embed/include/sys/_types.h:44: Error: unrecognized opcode `typedef long _off_t'
/home/francisco/opt/gnu-mcu-eclipse/riscv-none-gcc/8.2.0-2.2-20190521-0004/riscv-none-embed/include/sys/_types.h:50: Error: unrecognized opcode `typedef int __pid_t'
/home/francisco/opt/gnu-mcu-eclipse/riscv-none-gcc/8.2.0-2.2-20190521-0004/riscv-none-embed/include/sys/_types.h:54: Error: unrecognized opcode `typedef short __dev_t'
/home/francisco/opt/gnu-mcu-eclipse/riscv-none-gcc/8.2.0-2.2-20190521-0004/riscv-none-embed/include/sys/_types.h:58: Error: unrecognized opcode `typedef unsigned short __uid_t'
/home/francisco/opt/gnu-mcu-eclipse/riscv-none-gcc/8.2.0-2.2-20190521-0004/riscv-none-embed/include/sys/_types.h:61: Error: unrecognized opcode `typedef unsigned short __gid_t'
/home/francisco/opt/gnu-mcu-eclipse/riscv-none-gcc/8.2.0-2.2-20190521-0004/riscv-none-embed/include/sys/_types.h:65: Error: unrecognized opcode `typedef __uint32_t __id_t'
/home/francisco/opt/gnu-mcu-eclipse/riscv-none-gcc/8.2.0-2.2-20190521-0004/riscv-none-embed/include/sys/_types.h:73: Error: unrecognized opcode `typedef unsigned short __ino_t'
/home/francisco/opt/gnu-mcu-eclipse/riscv-none-gcc/8.2.0-2.2-20190521-0004/riscv-none-embed/include/sys/_types.h:88: Error: unrecognized opcode `typedef __uint32_t __mode_t'
/home/francisco/opt/gnu-mcu-eclipse/riscv-none-gcc/8.2.0-2.2-20190521-0004/riscv-none-embed/include/sys/_types.h:94: Error: unrecognized opcode `__extension__ typedef long long _off64_t'
/home/francisco/opt/gnu-mcu-eclipse/riscv-none-gcc/8.2.0-2.2-20190521-0004/riscv-none-embed/include/sys/_types.h:100: Error: unrecognized opcode `typedef _off_t __off_t'
/home/francisco/opt/gnu-mcu-eclipse/riscv-none-gcc/8.2.0-2.2-20190521-0004/riscv-none-embed/include/sys/_types.h:103: Error: unrecognized opcode `typedef _off64_t __loff_t'
/home/francisco/opt/gnu-mcu-eclipse/riscv-none-gcc/8.2.0-2.2-20190521-0004/riscv-none-embed/include/sys/_types.h:106: Error: unrecognized opcode `typedef long __key_t'
/home/francisco/opt/gnu-mcu-eclipse/riscv-none-gcc/8.2.0-2.2-20190521-0004/riscv-none-embed/include/sys/_types.h:114: Error: unrecognized opcode `typedef long _fpos_t'
/home/francisco/opt/gnu-mcu-eclipse/riscv-none-gcc/8.2.0-2.2-20190521-0004/riscv-none-embed/include/sys/_types.h:129: Error: unrecognized opcode `typedef unsigned int __size_t'
/home/francisco/opt/gnu-mcu-eclipse/riscv-none-gcc/8.2.0-2.2-20190521-0004/riscv-none-embed/include/sys/_types.h:145: Error: unrecognized opcode `typedef signed int _ssize_t'
/home/francisco/opt/gnu-mcu-eclipse/riscv-none-gcc/8.2.0-2.2-20190521-0004/riscv-none-embed/include/sys/_types.h:156: Error: unrecognized opcode `typedef _ssize_t __ssize_t'
/home/francisco/opt/gnu-mcu-eclipse/riscv-none-gcc/8.2.0-2.2-20190521-0004/lib/gcc/riscv-none-embed/8.2.0/include/stddef.h:357: Error: unrecognized opcode `typedef unsigned int wint_t'
/home/francisco/opt/gnu-mcu-eclipse/riscv-none-gcc/8.2.0-2.2-20190521-0004/riscv-none-embed/include/sys/_types.h:163: Error: unrecognized opcode `typedef struct'
/home/francisco/opt/gnu-mcu-eclipse/riscv-none-gcc/8.2.0-2.2-20190521-0004/riscv-none-embed/include/sys/_types.h:164: Error: junk at end of line, first unrecognized character is `{'
/home/francisco/opt/gnu-mcu-eclipse/riscv-none-gcc/8.2.0-2.2-20190521-0004/riscv-none-embed/include/sys/_types.h:165: Error: unrecognized opcode `int __count'
/home/francisco/opt/gnu-mcu-eclipse/riscv-none-gcc/8.2.0-2.2-20190521-0004/riscv-none-embed/include/sys/_types.h:166: Error: unrecognized opcode `union'
/home/francisco/opt/gnu-mcu-eclipse/riscv-none-gcc/8.2.0-2.2-20190521-0004/riscv-none-embed/include/sys/_types.h:167: Error: junk at end of line, first unrecognized character is `{'
/home/francisco/opt/gnu-mcu-eclipse/riscv-none-gcc/8.2.0-2.2-20190521-0004/riscv-none-embed/include/sys/_types.h:168: Error: unrecognized opcode `wint_t __wch'
/home/francisco/opt/gnu-mcu-eclipse/riscv-none-gcc/8.2.0-2.2-20190521-0004/riscv-none-embed/include/sys/_types.h:169: Error: unrecognized opcode `unsigned char __wchb[4]'
/home/francisco/opt/gnu-mcu-eclipse/riscv-none-gcc/8.2.0-2.2-20190521-0004/riscv-none-embed/include/sys/_types.h:170: Error: junk at end of line, first unrecognized character is `}'
/home/francisco/opt/gnu-mcu-eclipse/riscv-none-gcc/8.2.0-2.2-20190521-0004/riscv-none-embed/include/sys/_types.h:171: Error: junk at end of line, first unrecognized character is `}'
/home/francisco/opt/gnu-mcu-eclipse/riscv-none-gcc/8.2.0-2.2-20190521-0004/riscv-none-embed/include/sys/_types.h:175: Error: unrecognized opcode `typedef _LOCK_RECURSIVE_T _flock_t'
/home/francisco/opt/gnu-mcu-eclipse/riscv-none-gcc/8.2.0-2.2-20190521-0004/riscv-none-embed/include/sys/_types.h:180: Error: unrecognized opcode `typedef void*_iconv_t'
/home/francisco/opt/gnu-mcu-eclipse/riscv-none-gcc/8.2.0-2.2-20190521-0004/riscv-none-embed/include/sys/_types.h:187: Error: unrecognized opcode `typedef unsigned long __clock_t'
/home/francisco/opt/gnu-mcu-eclipse/riscv-none-gcc/8.2.0-2.2-20190521-0004/riscv-none-embed/include/sys/_types.h:194: Error: unrecognized opcode `typedef __int_least64_t __time_t'
/home/francisco/opt/gnu-mcu-eclipse/riscv-none-gcc/8.2.0-2.2-20190521-0004/riscv-none-embed/include/sys/_types.h:197: Error: unrecognized opcode `typedef unsigned long __clockid_t'
/home/francisco/opt/gnu-mcu-eclipse/riscv-none-gcc/8.2.0-2.2-20190521-0004/riscv-none-embed/include/sys/_types.h:200: Error: unrecognized opcode `typedef unsigned long __timer_t'
/home/francisco/opt/gnu-mcu-eclipse/riscv-none-gcc/8.2.0-2.2-20190521-0004/riscv-none-embed/include/sys/_types.h:203: Error: unrecognized opcode `typedef __uint8_t __sa_family_t'
/home/francisco/opt/gnu-mcu-eclipse/riscv-none-gcc/8.2.0-2.2-20190521-0004/riscv-none-embed/include/sys/_types.h:207: Error: unrecognized opcode `typedef __uint32_t __socklen_t'
/home/francisco/opt/gnu-mcu-eclipse/riscv-none-gcc/8.2.0-2.2-20190521-0004/riscv-none-embed/include/sys/_types.h:210: Error: unrecognized opcode `typedef unsigned short __nlink_t'
/home/francisco/opt/gnu-mcu-eclipse/riscv-none-gcc/8.2.0-2.2-20190521-0004/riscv-none-embed/include/sys/_types.h:211: Error: unrecognized opcode `typedef long __suseconds_t'
/home/francisco/opt/gnu-mcu-eclipse/riscv-none-gcc/8.2.0-2.2-20190521-0004/riscv-none-embed/include/sys/_types.h:212: Error: unrecognized opcode `typedef unsigned long __useconds_t'
/home/francisco/opt/gnu-mcu-eclipse/riscv-none-gcc/8.2.0-2.2-20190521-0004/riscv-none-embed/include/sys/_types.h:215: Error: unrecognized opcode `typedef __builtin_va_list __va_list'
/home/francisco/opt/gnu-mcu-eclipse/riscv-none-gcc/8.2.0-2.2-20190521-0004/riscv-none-embed/include/sys/_sigset.h:41: Error: unrecognized opcode `typedef unsigned long __sigset_t'
/home/francisco/opt/gnu-mcu-eclipse/riscv-none-gcc/8.2.0-2.2-20190521-0004/riscv-none-embed/include/sys/_timeval.h:35: Error: unrecognized opcode `typedef __suseconds_t suseconds_t'
/home/francisco/opt/gnu-mcu-eclipse/riscv-none-gcc/8.2.0-2.2-20190521-0004/riscv-none-embed/include/sys/_timeval.h:40: Error: unrecognized opcode `typedef __int_least64_t time_t'
/home/francisco/opt/gnu-mcu-eclipse/riscv-none-gcc/8.2.0-2.2-20190521-0004/riscv-none-embed/include/sys/_timeval.h:52: Error: unrecognized opcode `struct timeval{'
/home/francisco/opt/gnu-mcu-eclipse/riscv-none-gcc/8.2.0-2.2-20190521-0004/riscv-none-embed/include/sys/_timeval.h:53: Error: unrecognized opcode `time_t tv_sec'
/home/francisco/opt/gnu-mcu-eclipse/riscv-none-gcc/8.2.0-2.2-20190521-0004/riscv-none-embed/include/sys/_timeval.h:54: Error: unrecognized opcode `suseconds_t tv_usec'
/home/francisco/opt/gnu-mcu-eclipse/riscv-none-gcc/8.2.0-2.2-20190521-0004/riscv-none-embed/include/sys/_timeval.h:55: Error: junk at end of line, first unrecognized character is `}'
/home/francisco/opt/gnu-mcu-eclipse/riscv-none-gcc/8.2.0-2.2-20190521-0004/riscv-none-embed/include/sys/_timespec.h:45: Error: unrecognized opcode `struct timespec{'
/home/francisco/opt/gnu-mcu-eclipse/riscv-none-gcc/8.2.0-2.2-20190521-0004/riscv-none-embed/include/sys/_timespec.h:46: Error: unrecognized opcode `time_t tv_sec'
/home/francisco/opt/gnu-mcu-eclipse/riscv-none-gcc/8.2.0-2.2-20190521-0004/riscv-none-embed/include/sys/_timespec.h:47: Error: unrecognized opcode `long tv_nsec'
/home/francisco/opt/gnu-mcu-eclipse/riscv-none-gcc/8.2.0-2.2-20190521-0004/riscv-none-embed/include/sys/_timespec.h:48: Error: junk at end of line, first unrecognized character is `}'
/home/francisco/opt/gnu-mcu-eclipse/riscv-none-gcc/8.2.0-2.2-20190521-0004/riscv-none-embed/include/sys/timespec.h:58: Error: unrecognized opcode `struct itimerspec{'
/home/francisco/opt/gnu-mcu-eclipse/riscv-none-gcc/8.2.0-2.2-20190521-0004/riscv-none-embed/include/sys/timespec.h:59: Error: unrecognized opcode `struct timespec it_interval'
/home/francisco/opt/gnu-mcu-eclipse/riscv-none-gcc/8.2.0-2.2-20190521-0004/riscv-none-embed/include/sys/timespec.h:60: Error: unrecognized opcode `struct timespec it_value'
/home/francisco/opt/gnu-mcu-eclipse/riscv-none-gcc/8.2.0-2.2-20190521-0004/riscv-none-embed/include/sys/timespec.h:61: Error: junk at end of line, first unrecognized character is `}'
/home/francisco/opt/gnu-mcu-eclipse/riscv-none-gcc/8.2.0-2.2-20190521-0004/riscv-none-embed/include/sys/select.h:31: Error: unrecognized opcode `typedef __sigset_t sigset_t'
/home/francisco/opt/gnu-mcu-eclipse/riscv-none-gcc/8.2.0-2.2-20190521-0004/riscv-none-embed/include/sys/select.h:45: Error: unrecognized opcode `typedef unsigned long fd_mask'
/home/francisco/opt/gnu-mcu-eclipse/riscv-none-gcc/8.2.0-2.2-20190521-0004/riscv-none-embed/include/sys/select.h:53: Error: unrecognized opcode `typedef struct _types_fd_set{'
/home/francisco/opt/gnu-mcu-eclipse/riscv-none-gcc/8.2.0-2.2-20190521-0004/riscv-none-embed/include/sys/select.h:54: Error: unrecognized opcode `fd_mask fds_bits[(((64)+(((sizeof(fd_mask)*8))-1))/((sizeof(fd_mask)*8)))]'
/home/francisco/opt/gnu-mcu-eclipse/riscv-none-gcc/8.2.0-2.2-20190521-0004/riscv-none-embed/include/sys/select.h:55: Error: junk at end of line, first unrecognized character is `}'
/home/francisco/opt/gnu-mcu-eclipse/riscv-none-gcc/8.2.0-2.2-20190521-0004/riscv-none-embed/include/sys/select.h:73: Error: unrecognized opcode `int select(int __n,_types_fd_set*__readfds,_types_fd_set*__writefds,_types_fd_set*__exceptfds,struct timeval*__timeout)'
/home/francisco/opt/gnu-mcu-eclipse/riscv-none-gcc/8.2.0-2.2-20190521-0004/riscv-none-embed/include/sys/select.h:76: Error: unrecognized opcode `int pselect(int __n,_types_fd_set*__readfds,_types_fd_set*__writefds,_types_fd_set*__exceptfds,const struct timespec*__timeout,const sigset_t*__set)'
/home/francisco/opt/gnu-mcu-eclipse/riscv-none-gcc/8.2.0-2.2-20190521-0004/riscv-none-embed/include/sys/types.h:73: Error: unrecognized opcode `typedef __uint32_t in_addr_t'
/home/francisco/opt/gnu-mcu-eclipse/riscv-none-gcc/8.2.0-2.2-20190521-0004/riscv-none-embed/include/sys/types.h:78: Error: unrecognized opcode `typedef __uint16_t in_port_t'
/home/francisco/opt/gnu-mcu-eclipse/riscv-none-gcc/8.2.0-2.2-20190521-0004/riscv-none-embed/include/sys/types.h:87: Error: unrecognized opcode `typedef unsigned char u_char'
/home/francisco/opt/gnu-mcu-eclipse/riscv-none-gcc/8.2.0-2.2-20190521-0004/riscv-none-embed/include/sys/types.h:91: Error: unrecognized opcode `typedef unsigned short u_short'
/home/francisco/opt/gnu-mcu-eclipse/riscv-none-gcc/8.2.0-2.2-20190521-0004/riscv-none-embed/include/sys/types.h:95: Error: unrecognized opcode `typedef unsigned int u_int'
/home/francisco/opt/gnu-mcu-eclipse/riscv-none-gcc/8.2.0-2.2-20190521-0004/riscv-none-embed/include/sys/types.h:99: Error: unrecognized opcode `typedef unsigned long u_long'
/home/francisco/opt/gnu-mcu-eclipse/riscv-none-gcc/8.2.0-2.2-20190521-0004/riscv-none-embed/include/sys/types.h:107: Error: unrecognized opcode `typedef unsigned short ushort'
/home/francisco/opt/gnu-mcu-eclipse/riscv-none-gcc/8.2.0-2.2-20190521-0004/riscv-none-embed/include/sys/types.h:108: Error: unrecognized opcode `typedef unsigned int uint'
/home/francisco/opt/gnu-mcu-eclipse/riscv-none-gcc/8.2.0-2.2-20190521-0004/riscv-none-embed/include/sys/types.h:109: Error: unrecognized opcode `typedef unsigned long ulong'
/home/francisco/opt/gnu-mcu-eclipse/riscv-none-gcc/8.2.0-2.2-20190521-0004/riscv-none-embed/include/sys/types.h:113: Error: unrecognized opcode `typedef __blkcnt_t blkcnt_t'
/home/francisco/opt/gnu-mcu-eclipse/riscv-none-gcc/8.2.0-2.2-20190521-0004/riscv-none-embed/include/sys/types.h:118: Error: unrecognized opcode `typedef __blksize_t blksize_t'
/home/francisco/opt/gnu-mcu-eclipse/riscv-none-gcc/8.2.0-2.2-20190521-0004/riscv-none-embed/include/sys/types.h:123: Error: unrecognized opcode `typedef unsigned long clock_t'
/home/francisco/opt/gnu-mcu-eclipse/riscv-none-gcc/8.2.0-2.2-20190521-0004/riscv-none-embed/include/sys/types.h:135: Error: unrecognized opcode `typedef long daddr_t'
/home/francisco/opt/gnu-mcu-eclipse/riscv-none-gcc/8.2.0-2.2-20190521-0004/riscv-none-embed/include/sys/types.h:139: Error: unrecognized opcode `typedef char*caddr_t'
/home/francisco/opt/gnu-mcu-eclipse/riscv-none-gcc/8.2.0-2.2-20190521-0004/riscv-none-embed/include/sys/types.h:144: Error: unrecognized opcode `typedef __fsblkcnt_t fsblkcnt_t'
/home/francisco/opt/gnu-mcu-eclipse/riscv-none-gcc/8.2.0-2.2-20190521-0004/riscv-none-embed/include/sys/types.h:145: Error: unrecognized opcode `typedef __fsfilcnt_t fsfilcnt_t'
/home/francisco/opt/gnu-mcu-eclipse/riscv-none-gcc/8.2.0-2.2-20190521-0004/riscv-none-embed/include/sys/types.h:150: Error: unrecognized opcode `typedef __id_t id_t'
/home/francisco/opt/gnu-mcu-eclipse/riscv-none-gcc/8.2.0-2.2-20190521-0004/riscv-none-embed/include/sys/types.h:155: Error: unrecognized opcode `typedef __ino_t ino_t'
/home/francisco/opt/gnu-mcu-eclipse/riscv-none-gcc/8.2.0-2.2-20190521-0004/riscv-none-embed/include/sys/types.h:173: Error: unrecognized opcode `typedef __off_t off_t'
/home/francisco/opt/gnu-mcu-eclipse/riscv-none-gcc/8.2.0-2.2-20190521-0004/riscv-none-embed/include/sys/types.h:177: Error: unrecognized opcode `typedef __dev_t dev_t'
/home/francisco/opt/gnu-mcu-eclipse/riscv-none-gcc/8.2.0-2.2-20190521-0004/riscv-none-embed/include/sys/types.h:181: Error: unrecognized opcode `typedef __uid_t uid_t'
/home/francisco/opt/gnu-mcu-eclipse/riscv-none-gcc/8.2.0-2.2-20190521-0004/riscv-none-embed/include/sys/types.h:185: Error: unrecognized opcode `typedef __gid_t gid_t'
/home/francisco/opt/gnu-mcu-eclipse/riscv-none-gcc/8.2.0-2.2-20190521-0004/riscv-none-embed/include/sys/types.h:190: Error: unrecognized opcode `typedef __pid_t pid_t'
/home/francisco/opt/gnu-mcu-eclipse/riscv-none-gcc/8.2.0-2.2-20190521-0004/riscv-none-embed/include/sys/types.h:195: Error: unrecognized opcode `typedef __key_t key_t'
/home/francisco/opt/gnu-mcu-eclipse/riscv-none-gcc/8.2.0-2.2-20190521-0004/riscv-none-embed/include/sys/types.h:200: Error: unrecognized opcode `typedef _ssize_t ssize_t'
/home/francisco/opt/gnu-mcu-eclipse/riscv-none-gcc/8.2.0-2.2-20190521-0004/riscv-none-embed/include/sys/types.h:205: Error: unrecognized opcode `typedef __mode_t mode_t'
/home/francisco/opt/gnu-mcu-eclipse/riscv-none-gcc/8.2.0-2.2-20190521-0004/riscv-none-embed/include/sys/types.h:210: Error: unrecognized opcode `typedef __nlink_t nlink_t'
/home/francisco/opt/gnu-mcu-eclipse/riscv-none-gcc/8.2.0-2.2-20190521-0004/riscv-none-embed/include/sys/types.h:215: Error: unrecognized opcode `typedef __clockid_t clockid_t'
/home/francisco/opt/gnu-mcu-eclipse/riscv-none-gcc/8.2.0-2.2-20190521-0004/riscv-none-embed/include/sys/types.h:221: Error: unrecognized opcode `typedef __timer_t timer_t'
/home/francisco/opt/gnu-mcu-eclipse/riscv-none-gcc/8.2.0-2.2-20190521-0004/riscv-none-embed/include/sys/types.h:227: Error: unrecognized opcode `typedef __useconds_t useconds_t'
/home/francisco/opt/gnu-mcu-eclipse/riscv-none-gcc/8.2.0-2.2-20190521-0004/riscv-none-embed/include/sys/types.h:236: Error: unrecognized opcode `typedef __int64_t sbintime_t'
/home/francisco/opt/gnu-mcu-eclipse/riscv-none-gcc/8.2.0-2.2-20190521-0004/riscv-none-embed/include/sys/sched.h:48: Error: unrecognized opcode `struct sched_param{'
/home/francisco/opt/gnu-mcu-eclipse/riscv-none-gcc/8.2.0-2.2-20190521-0004/riscv-none-embed/include/sys/sched.h:49: Error: unrecognized opcode `int sched_priority'
/home/francisco/opt/gnu-mcu-eclipse/riscv-none-gcc/8.2.0-2.2-20190521-0004/riscv-none-embed/include/sys/sched.h:61: Error: junk at end of line, first unrecognized character is `}'
/home/francisco/opt/gnu-mcu-eclipse/riscv-none-gcc/8.2.0-2.2-20190521-0004/riscv-none-embed/include/sys/_pthreadtypes.h:32: Error: unrecognized opcode `typedef __uint32_t pthread_t'
/home/francisco/opt/gnu-mcu-eclipse/riscv-none-gcc/8.2.0-2.2-20190521-0004/riscv-none-embed/include/sys/_pthreadtypes.h:61: Error: unrecognized opcode `typedef struct{'
/home/francisco/opt/gnu-mcu-eclipse/riscv-none-gcc/8.2.0-2.2-20190521-0004/riscv-none-embed/include/sys/_pthreadtypes.h:62: Error: unrecognized opcode `int is_initialized'
/home/francisco/opt/gnu-mcu-eclipse/riscv-none-gcc/8.2.0-2.2-20190521-0004/riscv-none-embed/include/sys/_pthreadtypes.h:63: Error: unrecognized opcode `void *stackaddr'
/home/francisco/opt/gnu-mcu-eclipse/riscv-none-gcc/8.2.0-2.2-20190521-0004/riscv-none-embed/include/sys/_pthreadtypes.h:64: Error: unrecognized opcode `int stacksize'
/home/francisco/opt/gnu-mcu-eclipse/riscv-none-gcc/8.2.0-2.2-20190521-0004/riscv-none-embed/include/sys/_pthreadtypes.h:65: Error: unrecognized opcode `int contentionscope'
/home/francisco/opt/gnu-mcu-eclipse/riscv-none-gcc/8.2.0-2.2-20190521-0004/riscv-none-embed/include/sys/_pthreadtypes.h:66: Error: unrecognized opcode `int inheritsched'
/home/francisco/opt/gnu-mcu-eclipse/riscv-none-gcc/8.2.0-2.2-20190521-0004/riscv-none-embed/include/sys/_pthreadtypes.h:67: Error: unrecognized opcode `int schedpolicy'
/home/francisco/opt/gnu-mcu-eclipse/riscv-none-gcc/8.2.0-2.2-20190521-0004/riscv-none-embed/include/sys/_pthreadtypes.h:68: Error: unrecognized opcode `struct sched_param schedparam'
/home/francisco/opt/gnu-mcu-eclipse/riscv-none-gcc/8.2.0-2.2-20190521-0004/riscv-none-embed/include/sys/_pthreadtypes.h:74: Error: unrecognized opcode `int detachstate'
/home/francisco/opt/gnu-mcu-eclipse/riscv-none-gcc/8.2.0-2.2-20190521-0004/riscv-none-embed/include/sys/_pthreadtypes.h:75: Error: junk at end of line, first unrecognized character is `}'
/home/francisco/opt/gnu-mcu-eclipse/riscv-none-gcc/8.2.0-2.2-20190521-0004/riscv-none-embed/include/sys/_pthreadtypes.h:154: Error: unrecognized opcode `typedef __uint32_t pthread_mutex_t'
/home/francisco/opt/gnu-mcu-eclipse/riscv-none-gcc/8.2.0-2.2-20190521-0004/riscv-none-embed/include/sys/_pthreadtypes.h:156: Error: unrecognized opcode `typedef struct{'
/home/francisco/opt/gnu-mcu-eclipse/riscv-none-gcc/8.2.0-2.2-20190521-0004/riscv-none-embed/include/sys/_pthreadtypes.h:157: Error: unrecognized opcode `int is_initialized'
/home/francisco/opt/gnu-mcu-eclipse/riscv-none-gcc/8.2.0-2.2-20190521-0004/riscv-none-embed/include/sys/_pthreadtypes.h:168: Error: unrecognized opcode `int recursive'
/home/francisco/opt/gnu-mcu-eclipse/riscv-none-gcc/8.2.0-2.2-20190521-0004/riscv-none-embed/include/sys/_pthreadtypes.h:169: Error: junk at end of line, first unrecognized character is `}'
/home/francisco/opt/gnu-mcu-eclipse/riscv-none-gcc/8.2.0-2.2-20190521-0004/riscv-none-embed/include/sys/_pthreadtypes.h:176: Error: unrecognized opcode `typedef __uint32_t pthread_cond_t'
/home/francisco/opt/gnu-mcu-eclipse/riscv-none-gcc/8.2.0-2.2-20190521-0004/riscv-none-embed/include/sys/_pthreadtypes.h:180: Error: unrecognized opcode `typedef struct{'
/home/francisco/opt/gnu-mcu-eclipse/riscv-none-gcc/8.2.0-2.2-20190521-0004/riscv-none-embed/include/sys/_pthreadtypes.h:181: Error: unrecognized opcode `int is_initialized'
/home/francisco/opt/gnu-mcu-eclipse/riscv-none-gcc/8.2.0-2.2-20190521-0004/riscv-none-embed/include/sys/_pthreadtypes.h:182: Error: unrecognized opcode `clock_t clock'
/home/francisco/opt/gnu-mcu-eclipse/riscv-none-gcc/8.2.0-2.2-20190521-0004/riscv-none-embed/include/sys/_pthreadtypes.h:186: Error: junk at end of line, first unrecognized character is `}'
/home/francisco/opt/gnu-mcu-eclipse/riscv-none-gcc/8.2.0-2.2-20190521-0004/riscv-none-embed/include/sys/_pthreadtypes.h:190: Error: unrecognized opcode `typedef __uint32_t pthread_key_t'
/home/francisco/opt/gnu-mcu-eclipse/riscv-none-gcc/8.2.0-2.2-20190521-0004/riscv-none-embed/include/sys/_pthreadtypes.h:192: Error: unrecognized opcode `typedef struct{'
/home/francisco/opt/gnu-mcu-eclipse/riscv-none-gcc/8.2.0-2.2-20190521-0004/riscv-none-embed/include/sys/_pthreadtypes.h:193: Error: unrecognized opcode `int is_initialized'
/home/francisco/opt/gnu-mcu-eclipse/riscv-none-gcc/8.2.0-2.2-20190521-0004/riscv-none-embed/include/sys/_pthreadtypes.h:194: Error: unrecognized opcode `int init_executed'
/home/francisco/opt/gnu-mcu-eclipse/riscv-none-gcc/8.2.0-2.2-20190521-0004/riscv-none-embed/include/sys/_pthreadtypes.h:195: Error: junk at end of line, first unrecognized character is `}'
/home/francisco/workspace/RIOT2/core/include/kernel_types.h:83: Error: unrecognized opcode `typedef int16_t kernel_pid_t'
/home/francisco/workspace/RIOT2/core/include/kernel_types.h:92: Error: unrecognized opcode `static inline int pid_is_valid(kernel_pid_t pid)'
/home/francisco/workspace/RIOT2/core/include/kernel_types.h:93: Error: junk at end of line, first unrecognized character is `{'
/home/francisco/workspace/RIOT2/core/include/kernel_types.h:94: Error: unrecognized opcode `return (((0+1)<=pid)&&(pid<=((0+1)+32-1)))'
/home/francisco/workspace/RIOT2/core/include/kernel_types.h:95: Error: junk at end of line, first unrecognized character is `}'
/home/francisco/workspace/RIOT2/core/include/msg.h:184: Error: unrecognized opcode `typedef struct{'
/home/francisco/workspace/RIOT2/core/include/msg.h:185: Error: unrecognized opcode `kernel_pid_t sender_pid'
/home/francisco/workspace/RIOT2/core/include/msg.h:187: Error: unrecognized opcode `uint16_t type'
/home/francisco/workspace/RIOT2/core/include/msg.h:188: Error: unrecognized opcode `union {'
/home/francisco/workspace/RIOT2/core/include/msg.h:189: Error: unrecognized opcode `void *ptr'
/home/francisco/workspace/RIOT2/core/include/msg.h:190: Error: unrecognized opcode `uint32_t value'
/home/francisco/workspace/RIOT2/core/include/msg.h:191: Error: junk at end of line, first unrecognized character is `}'
/home/francisco/workspace/RIOT2/core/include/msg.h:192: Error: junk at end of line, first unrecognized character is `}'
/home/francisco/workspace/RIOT2/core/include/msg.h:213: Error: unrecognized opcode `int msg_send(msg_t*m,kernel_pid_t target_pid)'
/home/francisco/workspace/RIOT2/core/include/msg.h:232: Error: unrecognized opcode `int msg_try_send(msg_t*m,kernel_pid_t target_pid)'
/home/francisco/workspace/RIOT2/core/include/msg.h:248: Error: unrecognized opcode `int msg_send_to_self(msg_t*m)'
/home/francisco/workspace/RIOT2/core/include/msg.h:273: Error: unrecognized opcode `int msg_send_int(msg_t*m,kernel_pid_t target_pid)'
/home/francisco/workspace/RIOT2/core/include/msg.h:282: Error: unrecognized opcode `static inline int msg_sent_by_int(const msg_t*m)'
/home/francisco/workspace/RIOT2/core/include/msg.h:283: Error: junk at end of line, first unrecognized character is `{'
/home/francisco/workspace/RIOT2/core/include/msg.h:284: Error: unrecognized opcode `return (m->sender_pid==(((0+1)+32-1)+1))'
/home/francisco/workspace/RIOT2/core/include/msg.h:285: Error: junk at end of line, first unrecognized character is `}'
/home/francisco/workspace/RIOT2/core/include/msg.h:297: Error: unrecognized opcode `int msg_receive(msg_t*m)'
/home/francisco/workspace/RIOT2/core/include/msg.h:310: Error: unrecognized opcode `int msg_try_receive(msg_t*m)'
/home/francisco/workspace/RIOT2/core/include/msg.h:328: Error: unrecognized opcode `int msg_send_receive(msg_t*m,msg_t*reply,kernel_pid_t target_pid)'
/home/francisco/workspace/RIOT2/core/include/msg.h:341: Error: unrecognized opcode `int msg_reply(msg_t*m,msg_t*reply)'
/home/francisco/workspace/RIOT2/core/include/msg.h:355: Error: unrecognized opcode `int msg_reply_int(msg_t*m,msg_t*reply)'
/home/francisco/workspace/RIOT2/core/include/msg.h:363: Error: unrecognized opcode `int msg_avail(void)'
/home/francisco/workspace/RIOT2/core/include/msg.h:375: Error: unrecognized opcode `void msg_init_queue(msg_t*array,int num)'
/home/francisco/workspace/RIOT2/core/include/msg.h:380: Error: unrecognized opcode `void msg_queue_print(void)'
/home/francisco/workspace/RIOT2/core/include/native_sched.h:58: Error: unrecognized opcode `extern int sched_yield(void)'
/home/francisco/workspace/RIOT2/core/include/sched.h:96: Error: unrecognized opcode `typedef struct _thread thread_t'
/home/francisco/workspace/RIOT2/core/include/sched.h:104: Error: unrecognized opcode `typedef enum{'
/home/francisco/workspace/RIOT2/core/include/sched.h:105: Error: unrecognized opcode `status_stopped,'
/home/francisco/workspace/RIOT2/core/include/sched.h:106: Error: unrecognized opcode `status_zombie,'
/home/francisco/workspace/RIOT2/core/include/sched.h:107: Error: unrecognized opcode `status_sleeping,'
/home/francisco/workspace/RIOT2/core/include/sched.h:108: Error: unrecognized opcode `status_mutex_blocked,'
/home/francisco/workspace/RIOT2/core/include/sched.h:109: Error: unrecognized opcode `status_receive_blocked,'
/home/francisco/workspace/RIOT2/core/include/sched.h:110: Error: unrecognized opcode `status_send_blocked,'
/home/francisco/workspace/RIOT2/core/include/sched.h:111: Error: unrecognized opcode `status_reply_blocked,'
/home/francisco/workspace/RIOT2/core/include/sched.h:112: Error: unrecognized opcode `status_flag_blocked_any,'
/home/francisco/workspace/RIOT2/core/include/sched.h:113: Error: unrecognized opcode `status_flag_blocked_all,'
/home/francisco/workspace/RIOT2/core/include/sched.h:114: Error: unrecognized opcode `status_mbox_blocked,'
/home/francisco/workspace/RIOT2/core/include/sched.h:115: Error: unrecognized opcode `status_cond_blocked,'
/home/francisco/workspace/RIOT2/core/include/sched.h:116: Error: unrecognized opcode `status_running,'
/home/francisco/workspace/RIOT2/core/include/sched.h:117: Error: unrecognized opcode `status_pending,'
/home/francisco/workspace/RIOT2/core/include/sched.h:118: Error: unrecognized opcode `status_numof'
/home/francisco/workspace/RIOT2/core/include/sched.h:119: Error: junk at end of line, first unrecognized character is `}'
/home/francisco/workspace/RIOT2/core/include/sched.h:145: Error: unrecognized opcode `thread_t *sched_run(void)'
/home/francisco/workspace/RIOT2/core/include/sched.h:154: Error: unrecognized opcode `void sched_set_status(thread_t*process,thread_status_t status)'
/home/francisco/workspace/RIOT2/core/include/sched.h:168: Error: unrecognized opcode `void sched_switch(uint16_t other_prio)'
/home/francisco/workspace/RIOT2/core/include/sched.h:173: Error: unrecognized opcode `__attribute__((noreturn)) void cpu_switch_context_exit(void)'
/home/francisco/workspace/RIOT2/core/include/sched.h:179: Error: unrecognized opcode `extern volatile unsigned int sched_context_switch_request'
/home/francisco/workspace/RIOT2/core/include/sched.h:184: Error: unrecognized opcode `extern volatile thread_t*sched_threads[((0+1)+32-1)+1]'
/home/francisco/workspace/RIOT2/core/include/sched.h:189: Error: unrecognized opcode `extern volatile int sched_num_threads'
/home/francisco/workspace/RIOT2/core/include/sched.h:194: Error: unrecognized opcode `extern clist_node_t sched_runqueues[16]'
/home/francisco/workspace/RIOT2/core/include/sched.h:199: Error: unrecognized opcode `__attribute__((noreturn)) void sched_task_exit(void)'
/home/francisco/workspace/RIOT2/core/include/sched.h:214: Error: unrecognized opcode `void sched_arch_idle(void)'
/home/francisco/workspace/RIOT2/core/include/thread.h:149: Error: unrecognized opcode `typedef void*(*thread_task_func_t)(void*arg)'
/home/francisco/workspace/RIOT2/core/include/thread.h:154: Error: unrecognized opcode `struct _thread{'
/home/francisco/workspace/RIOT2/core/include/thread.h:155: Error: unrecognized opcode `char *sp'
/home/francisco/workspace/RIOT2/core/include/thread.h:156: Error: unrecognized opcode `thread_status_t status'
/home/francisco/workspace/RIOT2/core/include/thread.h:157: Error: unrecognized opcode `uint8_t priority'
/home/francisco/workspace/RIOT2/core/include/thread.h:159: Error: unrecognized opcode `kernel_pid_t pid'
/home/francisco/workspace/RIOT2/core/include/thread.h:165: Error: unrecognized opcode `clist_node_t rq_entry'
/home/francisco/workspace/RIOT2/core/include/thread.h:169: Error: unrecognized opcode `void *wait_data'
/home/francisco/workspace/RIOT2/core/include/thread.h:173: Error: unrecognized opcode `list_node_t msg_waiters'
/home/francisco/workspace/RIOT2/core/include/thread.h:176: Error: unrecognized opcode `cib_t msg_queue'
/home/francisco/workspace/RIOT2/core/include/thread.h:178: Error: unrecognized opcode `msg_t *msg_array'
/home/francisco/workspace/RIOT2/core/include/thread.h:183: Error: unrecognized opcode `char *stack_start'
/home/francisco/workspace/RIOT2/core/include/thread.h:186: Error: unrecognized opcode `const char*name'
/home/francisco/workspace/RIOT2/core/include/thread.h:189: Error: unrecognized opcode `int stack_size'
/home/francisco/workspace/RIOT2/core/include/thread.h:198: Error: junk at end of line, first unrecognized character is `}'
/home/francisco/workspace/RIOT2/core/include/thread.h:360: Error: unrecognized opcode `kernel_pid_t thread_create(char*stack,'
/home/francisco/workspace/RIOT2/core/include/thread.h:361: Error: unrecognized opcode `int stacksize,'
/home/francisco/workspace/RIOT2/core/include/thread.h:362: Error: unrecognized opcode `uint8_t priority,'
/home/francisco/workspace/RIOT2/core/include/thread.h:363: Error: unrecognized opcode `int flags,'
/home/francisco/workspace/RIOT2/core/include/thread.h:364: Error: unrecognized opcode `thread_task_func_t task_func,'
/home/francisco/workspace/RIOT2/core/include/thread.h:365: Error: unrecognized opcode `void *arg,'
/home/francisco/workspace/RIOT2/core/include/thread.h:366: Error: unrecognized opcode `const char*name)'
/home/francisco/workspace/RIOT2/core/include/thread.h:374: Error: unrecognized opcode `static inline thread_t*thread_get_unchecked(kernel_pid_t pid)'
/home/francisco/workspace/RIOT2/core/include/thread.h:375: Error: junk at end of line, first unrecognized character is `{'
/home/francisco/workspace/RIOT2/core/include/thread.h:376: Error: unrecognized opcode `return (thread_t*)sched_threads[pid]'
/home/francisco/workspace/RIOT2/core/include/thread.h:377: Error: junk at end of line, first unrecognized character is `}'
/home/francisco/workspace/RIOT2/core/include/thread.h:386: Error: unrecognized opcode `static inline thread_t*thread_get(kernel_pid_t pid)'
/home/francisco/workspace/RIOT2/core/include/thread.h:387: Error: junk at end of line, first unrecognized character is `{'
/home/francisco/workspace/RIOT2/core/include/thread.h:388: Error: unrecognized opcode `if (pid_is_valid(pid)){'
/home/francisco/workspace/RIOT2/core/include/thread.h:389: Error: unrecognized opcode `return thread_get_unchecked(pid)'
/home/francisco/workspace/RIOT2/core/include/thread.h:390: Error: junk at end of line, first unrecognized character is `}'
/home/francisco/workspace/RIOT2/core/include/thread.h:391: Error: unrecognized opcode `return ((void*)0)'
/home/francisco/workspace/RIOT2/core/include/thread.h:392: Error: junk at end of line, first unrecognized character is `}'
/home/francisco/workspace/RIOT2/core/include/thread.h:402: Error: unrecognized opcode `thread_status_t thread_getstatus(kernel_pid_t pid)'
/home/francisco/workspace/RIOT2/core/include/thread.h:407: Error: unrecognized opcode `void thread_sleep(void)'
/home/francisco/workspace/RIOT2/core/include/thread.h:420: Error: unrecognized opcode `void thread_yield(void)'
/home/francisco/workspace/RIOT2/core/include/thread.h:434: Error: unrecognized opcode `void thread_yield_higher(void)'
/home/francisco/workspace/RIOT2/core/include/thread.h:445: Error: unrecognized opcode `void thread_zombify(void)'
/home/francisco/workspace/RIOT2/core/include/thread.h:455: Error: unrecognized opcode `int thread_kill_zombie(kernel_pid_t pid)'
/home/francisco/workspace/RIOT2/core/include/thread.h:465: Error: unrecognized opcode `int thread_wakeup(kernel_pid_t pid)'
/home/francisco/workspace/RIOT2/core/include/thread.h:472: Error: unrecognized opcode `static inline kernel_pid_t thread_getpid(void)'
/home/francisco/workspace/RIOT2/core/include/thread.h:473: Error: junk at end of line, first unrecognized character is `{'
/home/francisco/workspace/RIOT2/core/include/thread.h:474: Error: unrecognized opcode `extern volatile kernel_pid_t sched_active_pid'
/home/francisco/workspace/RIOT2/core/include/thread.h:476: Error: unrecognized opcode `return sched_active_pid'
/home/francisco/workspace/RIOT2/core/include/thread.h:477: Error: junk at end of line, first unrecognized character is `}'
/home/francisco/workspace/RIOT2/core/include/thread.h:486: Error: unrecognized opcode `static inline thread_t*thread_get_active(void)'
/home/francisco/workspace/RIOT2/core/include/thread.h:487: Error: junk at end of line, first unrecognized character is `{'
/home/francisco/workspace/RIOT2/core/include/thread.h:488: Error: unrecognized opcode `extern volatile thread_t*sched_active_thread'
/home/francisco/workspace/RIOT2/core/include/thread.h:490: Error: unrecognized opcode `return (thread_t*)sched_active_thread'
/home/francisco/workspace/RIOT2/core/include/thread.h:491: Error: junk at end of line, first unrecognized character is `}'
/home/francisco/workspace/RIOT2/core/include/thread.h:503: Error: unrecognized opcode `char *thread_stack_init(thread_task_func_t task_func,void*arg,'
/home/francisco/workspace/RIOT2/core/include/thread.h:504: Error: unrecognized opcode `void *stack_start,int stack_size)'
/home/francisco/workspace/RIOT2/core/include/thread.h:519: Error: unrecognized opcode `void thread_add_to_list(list_node_t*list,thread_t*thread)'
/home/francisco/workspace/RIOT2/core/include/thread.h:531: Error: unrecognized opcode `const char*thread_getname(kernel_pid_t pid)'
/home/francisco/workspace/RIOT2/core/include/thread.h:543: Error: unrecognized opcode `uintptr_t thread_measure_stack_free(const char*stack)'
/home/francisco/workspace/RIOT2/core/include/thread.h:549: Error: unrecognized opcode `int thread_isr_stack_usage(void)'
/home/francisco/workspace/RIOT2/core/include/thread.h:554: Error: unrecognized opcode `void *thread_isr_stack_pointer(void)'
/home/francisco/workspace/RIOT2/core/include/thread.h:559: Error: unrecognized opcode `void *thread_isr_stack_start(void)'
/home/francisco/workspace/RIOT2/core/include/thread.h:564: Error: unrecognized opcode `void thread_stack_print(void)'
/home/francisco/workspace/RIOT2/core/include/thread.h:569: Error: unrecognized opcode `void thread_print_stack(void)'
/home/francisco/workspace/RIOT2/core/include/thread.h:581: Error: unrecognized opcode `static inline int thread_has_msg_queue(const volatile struct _thread*thread)'
/home/francisco/workspace/RIOT2/core/include/thread.h:582: Error: junk at end of line, first unrecognized character is `{'
/home/francisco/workspace/RIOT2/core/include/thread.h:584: Error: unrecognized opcode `return (thread->msg_array!=((void*)0))'
/home/francisco/workspace/RIOT2/core/include/thread.h:589: Error: junk at end of line, first unrecognized character is `}'
/home/francisco/workspace/RIOT2/cpu/fe310/include/vendor/plic_driver.h:7: Error: unrecognized opcode `typedef uint32_t plic_source'
/home/francisco/workspace/RIOT2/cpu/fe310/include/vendor/plic_driver.h:8: Error: unrecognized opcode `typedef uint32_t plic_priority'
/home/francisco/workspace/RIOT2/cpu/fe310/include/vendor/plic_driver.h:9: Error: unrecognized opcode `typedef uint32_t plic_threshold'
/home/francisco/workspace/RIOT2/cpu/fe310/include/vendor/plic_driver.h:12: Error: unrecognized opcode `void PLIC_init('
/home/francisco/workspace/RIOT2/cpu/fe310/include/vendor/plic_driver.h:13: Error: unrecognized opcode `uintptr_t base_addr,'
/home/francisco/workspace/RIOT2/cpu/fe310/include/vendor/plic_driver.h:14: Error: unrecognized opcode `uint32_t num_sources,'
/home/francisco/workspace/RIOT2/cpu/fe310/include/vendor/plic_driver.h:15: Error: unrecognized opcode `uint32_t num_priorities'
/home/francisco/workspace/RIOT2/cpu/fe310/include/vendor/plic_driver.h:16: Error: junk at end of line, first unrecognized character is `)'
/home/francisco/workspace/RIOT2/cpu/fe310/include/vendor/plic_driver.h:18: Error: unrecognized opcode `void PLIC_set_threshold(plic_threshold threshold)'
/home/francisco/workspace/RIOT2/cpu/fe310/include/vendor/plic_driver.h:20: Error: unrecognized opcode `void PLIC_enable_interrupt(plic_source source)'
/home/francisco/workspace/RIOT2/cpu/fe310/include/vendor/plic_driver.h:22: Error: unrecognized opcode `void PLIC_disable_interrupt(plic_source source)'
/home/francisco/workspace/RIOT2/cpu/fe310/include/vendor/plic_driver.h:24: Error: unrecognized opcode `void PLIC_set_priority('
/home/francisco/workspace/RIOT2/cpu/fe310/include/vendor/plic_driver.h:25: Error: unrecognized opcode `plic_source source,'
/home/francisco/workspace/RIOT2/cpu/fe310/include/vendor/plic_driver.h:26: Error: unrecognized opcode `plic_priority priority)'
/home/francisco/workspace/RIOT2/cpu/fe310/include/vendor/plic_driver.h:28: Error: unrecognized opcode `plic_source PLIC_claim_interrupt(void)'
/home/francisco/workspace/RIOT2/cpu/fe310/include/vendor/plic_driver.h:30: Error: unrecognized opcode `void PLIC_complete_interrupt(plic_source source)'
/home/francisco/workspace/RIOT2/cpu/fe310/include/cpu.h:41: Error: unrecognized opcode `void cpu_init(void)'
/home/francisco/workspace/RIOT2/cpu/fe310/include/cpu.h:46: Error: unrecognized opcode `void clock_init(void)'
/home/francisco/workspace/RIOT2/cpu/fe310/include/cpu.h:53: Error: unrecognized opcode `uint32_t cpu_freq(void)'
/home/francisco/workspace/RIOT2/cpu/fe310/include/cpu.h:58: Error: unrecognized opcode `void irq_init(void)'
/home/francisco/workspace/RIOT2/cpu/fe310/include/cpu.h:65: Error: unrecognized opcode `static inline void cpu_print_last_instruction(void)'
/home/francisco/workspace/RIOT2/cpu/fe310/include/cpu.h:66: Error: junk at end of line, first unrecognized character is `{'
/home/francisco/workspace/RIOT2/cpu/fe310/include/cpu.h:68: Error: junk at end of line, first unrecognized character is `}'
/home/francisco/workspace/RIOT2/cpu/fe310/include/cpu.h:73: Error: unrecognized opcode `void nanostubs_init(void)'
/home/francisco/workspace/RIOT2/cpu/fe310/include/periph_cpu.h:47: Error: unrecognized opcode `typedef uint8_t gpio_t'
/home/francisco/workspace/RIOT2/cpu/fe310/include/periph_cpu.h:68: Error: unrecognized opcode `typedef struct{'
/home/francisco/workspace/RIOT2/cpu/fe310/include/periph_cpu.h:69: Error: unrecognized opcode `uint32_t addr'
/home/francisco/workspace/RIOT2/cpu/fe310/include/periph_cpu.h:70: Error: unrecognized opcode `gpio_t rx'
/home/francisco/workspace/RIOT2/cpu/fe310/include/periph_cpu.h:71: Error: unrecognized opcode `gpio_t tx'
/home/francisco/workspace/RIOT2/cpu/fe310/include/periph_cpu.h:72: Error: unrecognized opcode `plic_source isr_num'
/home/francisco/workspace/RIOT2/cpu/fe310/include/periph_cpu.h:73: Error: junk at end of line, first unrecognized character is `}'
/home/francisco/workspace/RIOT2/cpu/fe310/include/periph_cpu.h:92: Error: unrecognized opcode `typedef struct{'
/home/francisco/workspace/RIOT2/cpu/fe310/include/periph_cpu.h:93: Error: unrecognized opcode `uint32_t addr'
/home/francisco/workspace/RIOT2/cpu/fe310/include/periph_cpu.h:94: Error: unrecognized opcode `gpio_t mosi'
/home/francisco/workspace/RIOT2/cpu/fe310/include/periph_cpu.h:95: Error: unrecognized opcode `gpio_t miso'
/home/francisco/workspace/RIOT2/cpu/fe310/include/periph_cpu.h:96: Error: unrecognized opcode `gpio_t sclk'
/home/francisco/workspace/RIOT2/cpu/fe310/include/periph_cpu.h:97: Error: junk at end of line, first unrecognized character is `}'
/home/francisco/workspace/RIOT2/cpu/fe310/include/periph_cpu.h:124: Error: unrecognized opcode `typedef enum{'
/home/francisco/workspace/RIOT2/cpu/fe310/include/periph_cpu.h:125: Error: unrecognized opcode `i2c_speed_normal,'
/home/francisco/workspace/RIOT2/cpu/fe310/include/periph_cpu.h:126: Error: unrecognized opcode `i2c_speed_fast,'
/home/francisco/workspace/RIOT2/cpu/fe310/include/periph_cpu.h:127: Error: junk at end of line, first unrecognized character is `}'
/home/francisco/workspace/RIOT2/cpu/fe310/include/periph_cpu.h:134: Error: unrecognized opcode `typedef struct{'
/home/francisco/workspace/RIOT2/cpu/fe310/include/periph_cpu.h:135: Error: unrecognized opcode `uint32_t addr'
/home/francisco/workspace/RIOT2/cpu/fe310/include/periph_cpu.h:136: Error: unrecognized opcode `gpio_t scl'
/home/francisco/workspace/RIOT2/cpu/fe310/include/periph_cpu.h:137: Error: unrecognized opcode `gpio_t sda'
/home/francisco/workspace/RIOT2/cpu/fe310/include/periph_cpu.h:138: Error: unrecognized opcode `i2c_speed_t speed'
/home/francisco/workspace/RIOT2/cpu/fe310/include/periph_cpu.h:139: Error: junk at end of line, first unrecognized character is `}'
/home/francisco/workspace/RIOT2/boards/hifive1b/include/periph_conf.h:133: Error: unrecognized opcode `static const uart_conf_t uart_config[]={'
/home/francisco/workspace/RIOT2/boards/hifive1b/include/periph_conf.h:134: Error: junk at end of line, first unrecognized character is `{'
/home/francisco/workspace/RIOT2/boards/hifive1b/include/periph_conf.h:135: Error: junk at end of line, first unrecognized character is `,'
/home/francisco/workspace/RIOT2/boards/hifive1b/include/periph_conf.h:136: Error: junk at end of line, first unrecognized character is `,'
/home/francisco/workspace/RIOT2/boards/hifive1b/include/periph_conf.h:137: Error: junk at end of line, first unrecognized character is `,'
/home/francisco/workspace/RIOT2/boards/hifive1b/include/periph_conf.h:138: Error: junk at end of line, first unrecognized character is `,'
/home/francisco/workspace/RIOT2/boards/hifive1b/include/periph_conf.h:139: Error: junk at end of line, first unrecognized character is `}'
/home/francisco/workspace/RIOT2/boards/hifive1b/include/periph_conf.h:140: Error: junk at end of line, first unrecognized character is `{'
/home/francisco/workspace/RIOT2/boards/hifive1b/include/periph_conf.h:141: Error: junk at end of line, first unrecognized character is `,'
/home/francisco/workspace/RIOT2/boards/hifive1b/include/periph_conf.h:142: Error: junk at end of line, first unrecognized character is `,'
/home/francisco/workspace/RIOT2/boards/hifive1b/include/periph_conf.h:143: Error: junk at end of line, first unrecognized character is `,'
/home/francisco/workspace/RIOT2/boards/hifive1b/include/periph_conf.h:144: Error: junk at end of line, first unrecognized character is `,'
/home/francisco/workspace/RIOT2/boards/hifive1b/include/periph_conf.h:145: Error: junk at end of line, first unrecognized character is `}'
/home/francisco/workspace/RIOT2/boards/hifive1b/include/periph_conf.h:146: Error: junk at end of line, first unrecognized character is `}'
/home/francisco/workspace/RIOT2/boards/hifive1b/include/periph_conf.h:156: Error: unrecognized opcode `static const spi_conf_t spi_config[]={'
/home/francisco/workspace/RIOT2/boards/hifive1b/include/periph_conf.h:157: Error: junk at end of line, first unrecognized character is `{'
/home/francisco/workspace/RIOT2/boards/hifive1b/include/periph_conf.h:158: Error: junk at end of line, first unrecognized character is `,'
/home/francisco/workspace/RIOT2/boards/hifive1b/include/periph_conf.h:159: Error: junk at end of line, first unrecognized character is `,'
/home/francisco/workspace/RIOT2/boards/hifive1b/include/periph_conf.h:160: Error: junk at end of line, first unrecognized character is `,'
/home/francisco/workspace/RIOT2/boards/hifive1b/include/periph_conf.h:161: Error: junk at end of line, first unrecognized character is `,'
/home/francisco/workspace/RIOT2/boards/hifive1b/include/periph_conf.h:162: Error: junk at end of line, first unrecognized character is `}'
/home/francisco/workspace/RIOT2/boards/hifive1b/include/periph_conf.h:163: Error: junk at end of line, first unrecognized character is `}'
/home/francisco/workspace/RIOT2/boards/hifive1b/include/periph_conf.h:180: Error: unrecognized opcode `static const i2c_conf_t i2c_config[]={'
/home/francisco/workspace/RIOT2/boards/hifive1b/include/periph_conf.h:181: Error: junk at end of line, first unrecognized character is `{'
/home/francisco/workspace/RIOT2/boards/hifive1b/include/periph_conf.h:182: Error: junk at end of line, first unrecognized character is `,'
/home/francisco/workspace/RIOT2/boards/hifive1b/include/periph_conf.h:183: Error: junk at end of line, first unrecognized character is `,'
/home/francisco/workspace/RIOT2/boards/hifive1b/include/periph_conf.h:184: Error: junk at end of line, first unrecognized character is `,'
/home/francisco/workspace/RIOT2/boards/hifive1b/include/periph_conf.h:185: Error: junk at end of line, first unrecognized character is `,'
/home/francisco/workspace/RIOT2/boards/hifive1b/include/periph_conf.h:186: Error: junk at end of line, first unrecognized character is `}'
/home/francisco/workspace/RIOT2/boards/hifive1b/include/periph_conf.h:187: Error: junk at end of line, first unrecognized character is `}'
/home/francisco/workspace/RIOT2/drivers/include/periph/gpio.h:118: Error: unrecognized opcode `typedef enum{'
/home/francisco/workspace/RIOT2/drivers/include/periph/gpio.h:119: Error: unrecognized opcode `gpio_in ,'
/home/francisco/workspace/RIOT2/drivers/include/periph/gpio.h:120: Error: unrecognized opcode `gpio_in_pd,'
/home/francisco/workspace/RIOT2/drivers/include/periph/gpio.h:121: Error: unrecognized opcode `gpio_in_pu,'
/home/francisco/workspace/RIOT2/drivers/include/periph/gpio.h:122: Error: unrecognized opcode `gpio_out,'
/home/francisco/workspace/RIOT2/drivers/include/periph/gpio.h:123: Error: unrecognized opcode `gpio_od,'
/home/francisco/workspace/RIOT2/drivers/include/periph/gpio.h:125: Error: unrecognized opcode `gpio_od_pu'
/home/francisco/workspace/RIOT2/drivers/include/periph/gpio.h:127: Error: junk at end of line, first unrecognized character is `}'
/home/francisco/workspace/RIOT2/drivers/include/periph/gpio.h:134: Error: unrecognized opcode `typedef enum{'
/home/francisco/workspace/RIOT2/drivers/include/periph/gpio.h:135: Error: junk at end of line, first unrecognized character is `,'
/home/francisco/workspace/RIOT2/drivers/include/periph/gpio.h:136: Error: junk at end of line, first unrecognized character is `,'
/home/francisco/workspace/RIOT2/drivers/include/periph/gpio.h:138: Error: junk at end of line, first unrecognized character is `}'
/home/francisco/workspace/RIOT2/drivers/include/periph/gpio.h:146: Error: unrecognized opcode `typedef void(*gpio_cb_t)(void*arg)'
/home/francisco/workspace/RIOT2/drivers/include/periph/gpio.h:152: Error: unrecognized opcode `typedef struct{'
/home/francisco/workspace/RIOT2/drivers/include/periph/gpio.h:153: Error: unrecognized opcode `gpio_cb_t cb'
/home/francisco/workspace/RIOT2/drivers/include/periph/gpio.h:154: Error: unrecognized opcode `void *arg'
/home/francisco/workspace/RIOT2/drivers/include/periph/gpio.h:155: Error: junk at end of line, first unrecognized character is `}'
/home/francisco/workspace/RIOT2/drivers/include/periph/gpio.h:171: Error: unrecognized opcode `int gpio_init(gpio_t pin,gpio_mode_t mode)'
/home/francisco/workspace/RIOT2/drivers/include/periph/gpio.h:230: Error: unrecognized opcode `int gpio_read(gpio_t pin)'
/home/francisco/workspace/RIOT2/drivers/include/periph/gpio.h:237: Error: unrecognized opcode `void gpio_set(gpio_t pin)'
/home/francisco/workspace/RIOT2/drivers/include/periph/gpio.h:244: Error: unrecognized opcode `void gpio_clear(gpio_t pin)'
/home/francisco/workspace/RIOT2/drivers/include/periph/gpio.h:251: Error: unrecognized opcode `void gpio_toggle(gpio_t pin)'
/home/francisco/workspace/RIOT2/drivers/include/periph/gpio.h:259: Error: unrecognized opcode `void gpio_write(gpio_t pin,int value)'
/home/francisco/workspace/RIOT2/drivers/include/periph/gpio.h:267: Error: unrecognized opcode `static inline int gpio_is_equal(gpio_t gpio1,gpio_t gpio2)'
/home/francisco/workspace/RIOT2/drivers/include/periph/gpio.h:268: Error: junk at end of line, first unrecognized character is `{'
/home/francisco/workspace/RIOT2/drivers/include/periph/gpio.h:269: Error: unrecognized opcode `return (gpio1==gpio2)'
/home/francisco/workspace/RIOT2/drivers/include/periph/gpio.h:270: Error: junk at end of line, first unrecognized character is `}'
/home/francisco/workspace/RIOT2/drivers/include/periph/gpio.h:277: Error: unrecognized opcode `static inline int gpio_is_valid(gpio_t gpio)'
/home/francisco/workspace/RIOT2/drivers/include/periph/gpio.h:278: Error: junk at end of line, first unrecognized character is `{'
/home/francisco/workspace/RIOT2/drivers/include/periph/gpio.h:279: Error: unrecognized opcode `return (gpio!=(0xff))'
/home/francisco/workspace/RIOT2/drivers/include/periph/gpio.h:280: Error: junk at end of line, first unrecognized character is `}'
/home/francisco/workspace/RIOT2/sys/include/dbgpin.h:38: Error: unrecognized opcode `extern const gpio_t dbgpin_pins[]'
/home/francisco/workspace/RIOT2/sys/include/dbgpin.h:46: Error: unrecognized opcode `static inline void dbgpin_set(unsigned pin)'
/home/francisco/workspace/RIOT2/sys/include/dbgpin.h:47: Error: junk at end of line, first unrecognized character is `{'
/home/francisco/workspace/RIOT2/sys/include/dbgpin.h:48: Error: unrecognized opcode `gpio_set(dbgpin_pins[pin])'
/home/francisco/workspace/RIOT2/sys/include/dbgpin.h:49: Error: junk at end of line, first unrecognized character is `}'
/home/francisco/workspace/RIOT2/sys/include/dbgpin.h:57: Error: unrecognized opcode `static inline void dbgpin_clear(unsigned pin)'
/home/francisco/workspace/RIOT2/sys/include/dbgpin.h:58: Error: junk at end of line, first unrecognized character is `{'
/home/francisco/workspace/RIOT2/sys/include/dbgpin.h:59: Error: unrecognized opcode `gpio_clear(dbgpin_pins[pin])'
/home/francisco/workspace/RIOT2/sys/include/dbgpin.h:60: Error: junk at end of line, first unrecognized character is `}'
/home/francisco/workspace/RIOT2/sys/include/dbgpin.h:68: Error: unrecognized opcode `static inline void dbgpin_toggle(unsigned pin)'
/home/francisco/workspace/RIOT2/sys/include/dbgpin.h:69: Error: junk at end of line, first unrecognized character is `{'
/home/francisco/workspace/RIOT2/sys/include/dbgpin.h:70: Error: unrecognized opcode `gpio_toggle(dbgpin_pins[pin])'
/home/francisco/workspace/RIOT2/sys/include/dbgpin.h:71: Error: junk at end of line, first unrecognized character is `}'
/home/francisco/workspace/RIOT2/sys/include/dbgpin.h:79: Error: unrecognized opcode `static inline void dbgpin_pulse(unsigned pin)'
/home/francisco/workspace/RIOT2/sys/include/dbgpin.h:80: Error: junk at end of line, first unrecognized character is `{'
/home/francisco/workspace/RIOT2/sys/include/dbgpin.h:81: Error: unrecognized opcode `dbgpin_toggle(pin)'
/home/francisco/workspace/RIOT2/sys/include/dbgpin.h:82: Error: unrecognized opcode `dbgpin_toggle(pin)'
/home/francisco/workspace/RIOT2/sys/include/dbgpin.h:83: Error: junk at end of line, first unrecognized character is `}'
/home/francisco/workspace/RIOT2/sys/include/dbgpin.h:92: Error: unrecognized opcode `static inline void dbgpin_signal(unsigned pin,unsigned num)'
/home/francisco/workspace/RIOT2/sys/include/dbgpin.h:93: Error: junk at end of line, first unrecognized character is `{'
/home/francisco/workspace/RIOT2/sys/include/dbgpin.h:94: Error: unrecognized opcode `for (unsigned i=0'
/home/francisco/workspace/RIOT2/sys/include/dbgpin.h:94: Error: unrecognized opcode `i <num'
/home/francisco/workspace/RIOT2/sys/include/dbgpin.h:94: Error: unrecognized opcode `i++) {'
/home/francisco/workspace/RIOT2/sys/include/dbgpin.h:95: Error: unrecognized opcode `dbgpin_pulse(pin)'
/home/francisco/workspace/RIOT2/sys/include/dbgpin.h:96: Error: junk at end of line, first unrecognized character is `}'
/home/francisco/workspace/RIOT2/sys/include/dbgpin.h:97: Error: junk at end of line, first unrecognized character is `}'
/home/francisco/workspace/RIOT2/sys/include/dbgpin.h:104: Error: unrecognized opcode `unsigned dbgpin_count(void)'
/home/francisco/workspace/RIOT2/sys/include/dbgpin.h:109: Error: unrecognized opcode `void dbgpin_init(void)'
/tmp/ccmbRPZZ.s: Error: symbol definition loop encountered at `list'
/tmp/ccmbRPZZ.s: Error: symbol definition loop encountered at `pos'
/tmp/ccmbRPZZ.s: Error: symbol definition loop encountered at `node'
make[2]: *** [/home/francisco/workspace/RIOT2/Makefile.base:140: /home/francisco/workspace/RIOT2/tests/dbgpin/bin/hifive1b/cpu/start.o] Error 1
make[1]: *** [/home/francisco/workspace/RIOT2/Makefile.base:30: ALL--/home/francisco/workspace/RIOT2/cpu/fe310] Error 2
make[1]: *** Waiting for unfinished jobs....
make: *** [/home/francisco/workspace/RIOT2/tests/dbgpin/../../Makefile.include:603: application_tests_dbgpin.module] Error 2
```
</details>

Reply:
Had the same issue with `esp32`